<!DOCTYPE html>

<html class="no-js" lang="en-US">

	<head>

		<meta charset="UTF-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">

		
		
				

		<title>PROGRAM &#8211; RECOMB 2019</title>
<link rel='dns-prefetch' href='//www.google.com' />
<link rel='dns-prefetch' href='//fonts.googleapis.com' />
<link rel='dns-prefetch' href='//s.w.org' />
<link rel="alternate" type="application/rss+xml" title="RECOMB 2019 &raquo; Feed" href="feed/" />
<link rel="alternate" type="application/rss+xml" title="RECOMB 2019 &raquo; Comments Feed" href="comments/feed/" />
		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11\/72x72\/","ext":".png","svgUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11\/svg\/","svgExt":".svg","source":{"concatemoji":"https:\/\/recomb2019.org\/wp-includes\/js\/wp-emoji-release.min.js?ver=5.0.10"}};
			!function(a,b,c){function d(a,b){var c=String.fromCharCode;l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,a),0,0);var d=k.toDataURL();l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,b),0,0);var e=k.toDataURL();return d===e}function e(a){var b;if(!l||!l.fillText)return!1;switch(l.textBaseline="top",l.font="600 32px Arial",a){case"flag":return!(b=d([55356,56826,55356,56819],[55356,56826,8203,55356,56819]))&&(b=d([55356,57332,56128,56423,56128,56418,56128,56421,56128,56430,56128,56423,56128,56447],[55356,57332,8203,56128,56423,8203,56128,56418,8203,56128,56421,8203,56128,56430,8203,56128,56423,8203,56128,56447]),!b);case"emoji":return b=d([55358,56760,9792,65039],[55358,56760,8203,9792,65039]),!b}return!1}function f(a){var c=b.createElement("script");c.src=a,c.defer=c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var g,h,i,j,k=b.createElement("canvas"),l=k.getContext&&k.getContext("2d");for(j=Array("flag","emoji"),c.supports={everything:!0,everythingExceptFlag:!0},i=0;i<j.length;i++)c.supports[j[i]]=e(j[i]),c.supports.everything=c.supports.everything&&c.supports[j[i]],"flag"!==j[i]&&(c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&c.supports[j[i]]);c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&!c.supports.flag,c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.everything||(h=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",h,!1),a.addEventListener("load",h,!1)):(a.attachEvent("onload",h),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),g=c.source||{},g.concatemoji?f(g.concatemoji):g.wpemoji&&g.twemoji&&(f(g.twemoji),f(g.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
<link rel='stylesheet' id='wp-block-library-css'  href='wp-includes/css/dist/block-library/style.min.css?ver=5.0.10' type='text/css' media='all' />
<link rel='stylesheet' id='contact-form-7-css'  href='wp-content/plugins/contact-form-7/includes/css/styles.css?ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='mptt-style-css'  href='wp-content/plugins/mp-timetable/media/css/style.css?ver=2.2.1' type='text/css' media='all' />
<link rel='stylesheet' id='parent-style-css'  href='wp-content/themes/Retro-4/style.css?ver=5.0.10' type='text/css' media='all' />
<link rel='stylesheet' id='child-style-css'  href='wp-content/themes/Retro-4-child/style.css?ver=6.3.5' type='text/css' media='all' />
<link rel='stylesheet' id='retro-fonts-css'  href='https://fonts.googleapis.com/css?family=Roboto+Slab%3A400%2C700%7COswald%3A400%2C700&#038;subset&#038;ver=5.0.10' type='text/css' media='all' />
<link rel='stylesheet' id='retro-icons-css'  href='wp-content/themes/Retro-4/fonts/retro-icons.css?ver=5.0.10' type='text/css' media='all' />
<link rel='stylesheet' id='retro-style-css'  href='wp-content/themes/Retro-4-child/style.css?ver=5.0.10' type='text/css' media='all' />
<link rel='stylesheet' id='tablepress-default-css'  href='wp-content/plugins/tablepress/css/default.min.css?ver=1.9.2' type='text/css' media='all' />
<link rel='stylesheet' id='tmm-css'  href='wp-content/plugins/team-members/inc/css/tmm_style.min.css?ver=5.0.10' type='text/css' media='all' />
<script type='text/javascript' src='wp-includes/js/jquery/jquery.js?ver=1.12.4'></script>
<script type='text/javascript' src='wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1'></script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/modernizr.js?ver=2.7.1'></script>
<link rel='https://api.w.org/' href='wp-json/' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="xmlrpc.php?rsd" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="wp-includes/wlwmanifest.xml" /> 
<meta name="generator" content="WordPress 5.0.10" />
<link rel="canonical" href="program/" />
<link rel='shortlink' href='?p=333' />
<link rel="alternate" type="application/json+oembed" href="wp-json/oembed/1.0/embed?url=https%3A%2F%2Frecomb2019.org%2Fprogram%2F" />
<link rel="alternate" type="text/xml+oembed" href="wp-json/oembed/1.0/embed?url=https%3A%2F%2Frecomb2019.org%2Fprogram%2F&#038;format=xml" />
<noscript><style type="text/css">.mptt-shortcode-wrapper .mptt-shortcode-table:first-of-type{display:table!important}.mptt-shortcode-wrapper .mptt-shortcode-table .mptt-event-container:hover{height:auto!important;min-height:100%!important}body.mprm_ie_browser .mptt-shortcode-wrapper .mptt-event-container{height:auto!important}@media (max-width:767px){.mptt-shortcode-wrapper .mptt-shortcode-table:first-of-type{display:none!important}}</style></noscript><style>body, input, textarea, .blog-list .post-title h3, .blog-list h4.post-meta, .portfolio-list ul li h3, .single h2.post-title, .page h2.post-title { font-family: "Roboto Slab", sans-serif; }</style><style>.hentry-content h1, .hentry-content h2, .hentry-content h3, .hentry-content h4, .hentry-content h5, .hentry-content h6, .section-subtitle, .blog-list h4.post-meta, .single h4.post-meta, .widget h3, .comments h3, .comments h4, .comment-author, .more-posts { font-family: "Oswald", serif; }</style><link rel="icon" href="wp-content/uploads/2018/09/RECOMB2019thumbnail-150x150-1-80x80.png" sizes="32x32" />
<link rel="icon" href="wp-content/uploads/2018/09/RECOMB2019thumbnail-150x150-1.png" sizes="192x192" />
<link rel="apple-touch-icon-precomposed" href="wp-content/uploads/2018/09/RECOMB2019thumbnail-150x150-1.png" />
<meta name="msapplication-TileImage" content="wp-content/uploads/2018/09/RECOMB2019thumbnail-150x150-1.png" />
		<style type="text/css" id="wp-custom-css">
			.tablepress-id-7 .column-1 {
	min-width: 30pt;
	width: 8%;
	max-width: 40pt;
}

.tablepress-id-7 .column-2 {
	width: 52%;
}

.tablepress-id-7 .column-3 {
	width: 40%;
}		</style>
	
		<style>

@font-face {
  font-family: "BazarMedium";
  src: url("wp-content/themes/Retro-4/fonts/bazarmedium.eot");
  src: url("wp-content/themes/Retro-4/fonts/bazarmedium.eot?#iefix") format("embedded-opentype"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.woff") format("woff"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.ttf") format("truetype"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.svg#BazarMedium") format("svg");
  font-weight: normal;
  font-style: normal;
}

/* Fix ugly font rendering */
@media screen and (-webkit-min-device-pixel-ratio:0) {
    @font-face {
        font-family: "BazarMedium";
            src: url("wp-content/themes/Retro-4/fonts/bazarmedium.svg#BazarMedium") format("svg");
    }
}

h1, h2, h3, h4, h5, h6,
nav ul#main-nav li {
  font-family: "BazarMedium";
  text-transform: uppercase;
}




	#logo a {

		background-image: url(wp-content/uploads/2018/11/RECOMB_plain-logo-tan.png);

	}






@import url('http://fonts.googleapis.com/css?family=PT+Sans:400,700');

.sticky-note{
  overflow-y: auto;
}
</style>
	</head>

	<body id="start" class="page-template-default page page-id-333 fixed-header left-logo">
<header role="banner">

    <div class="header-inner " style="background-color: #004065">

        <div class="container">

            <div class="row clear">

                
<div id="logo">

	<a href="index.html" title="RECOMB 2019" >

		<h1>RECOMB 2019</h1>

	</a>
	
</div>
                <nav role="navigation">

<ul id="main-nav" class="clear"><li id="menu-item-975" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="awards.html">AWARDS</a></li>
<li id="menu-item-26" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="keynotes.html">Keynotes</a></li>
<li id="menu-item-800" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="schedule.html">SCHEDULE</a></li>
<li id="menu-item-853" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="materials.html">MATERIALS</a></li>
<li id="menu-item-921" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="venue.html">VENUE</a></li>
<li id="menu-item-28" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="committees.html">COMMITTEES</a></li>
<li id="menu-item-468" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="travel-fellowship.html">Fellowship</a></li>
<li id="menu-item-320" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="sponsors.html">SPONSORS</a></li>
<li id="menu-item-307" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="contact-form.html">Contact Us</a></li>
</ul>

                </nav>

            </div><!-- row -->

        </div><!-- container -->

        <hr class="bottom-dashed">  

    </div><!-- header-inner -->   

</header>
			

	
	<main role="main">

		<section>

			<div class="section-inner section-post " style="background-color: #fffffe">
				
				<hr class="top-dashed"> 

				<div class="container">

					<div class="row clear">

						<article class="col col-12 tablet-full mobile-full">

							<h2 class="post-title">PROGRAM</h2>

							<hr class="single-hr">

							<div class="hentry-content">
								<ul>
<li><a href="#Key">Keynote Talks</a></li>
<li><a href="#Acc">Accepted Papers</a></li>
<li><a href="#High">Highlight Talks</a></li>
<li><a href="#AccPost">Accepted Posters</a></li>
<li><a href="#LatePost">Accepted Late Breaking Posters</a></li>
</ul>
<p><center></p>
<h2 id="Acc">Keynote Talks</h2>
<p></center></p>
<ol>
<li>
<div class="paper"><span class="authors">Alfonso Valencia.</span><span class="title"><i>The Landscape of Disease Comorbidities</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Franziska Michor.</span><span class="title"><i>Evolutionary mathematical modeling enables optimization of glioblastoma treatment strategies</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Eytan Ruppin.</span><span class="title"><i>Harnessing genetic interactions to advance whole-exome precision cancer treatment</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Rachel Kolodny.</span><span class="title"><i>Evolution-CSI: hunting for evidence in the protein universe</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Carlos Bustamante.</span><span class="title"><i>Innovating in Biomedical Data Science for Precision Health</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Mihai Pop.</span><span class="title"><i>Uncovering heterogeneity in microbial communities</i></span></div>
</li>
</ol>
<p><center></p>
<h2 id="Acc">Accepted Papers</h2>
<p></center></p>
<ol>
<li>
<div class="paper"><span class="authors">Fatemeh Almodaresi, Prashant Pandey, Michael Ferdman, Rob Johnson and Rob Patro.</span><span class="title"><i>An Efficient and Scalable Representation of High-Dimensional Color Information Enabled via de Bruijn Graph Search</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Aryan Arbabi, David Adams, Sanja Fidler and Michael Brudno. </span><span class="title">Identifying clinical terms in free-text notes using ontology-guided machine learning</span></div>
</li>
<li>
<div class="paper"><span class="authors">Metin Balaban, Shahab Sarmashghi and Siavash Mirarab. </span><span class="title">APPLES: Fast Distance Based Phylogenetic Placement</span></div>
</li>
<li>
<div class="paper"><span class="authors">Bahar Behsaz, Hosein Mohimani, Alexey Gurevich, Andrey Prjibelski, Mark F Fisher, Larry Smarr, Pieter C. Dorrestein, Joshua S. Mylne and Pavel A. Pevzner. </span><span class="title">De Novo Peptide Sequencing Reveals a Vast Cyclopeptidome in Human Gut and Other Environments</span></div>
</li>
<li>
<div class="paper"><span class="authors">Philipp Benner and Martin Vingron. </span><span class="title"><i>ModHMM: A modular supra-Bayesian genome segmentation method</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Lodewijk Brand, Xue Yang, Liu Kai, Saad Elbeleidy, Hua Wang and Hao Zhang. </span><span class="title"><i>Learning Robust Multi-Label Sample Specific Distances for Identifying HIV-1 Drug Resistance</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Dexiong Chen, Laurent Jacob and Julien Mairal. </span><span class="title">Biological Sequence Modeling with Convolutional Kernel Networks</span></div>
</li>
<li>
<div class="paper"><span class="authors">Van Hoan Do, Mislav Blažević, Pablo Monteagudo, Luka Borozan, Khaled Elbassioni, Soeren Laue, Francisca Rojas Ringeling, Domagoj Matijevic and Stefan Canzar. </span><span class="title">Dynamic pseudo-time warping of complex single-cell trajectories</span></div>
</li>
<li>
<div class="paper"><span class="authors">Rebecca Elyanow, Bianca Dumitrascu, Barbara E. Engelhardt, and Benjamin J. Raphael. </span><span class="title">netNMF: A network regularization algorithm for dimensionality reduction and imputation of single-cell expression data</span></div>
</li>
<li>
<div class="paper"><span class="authors">Boying Gong and Elizabeth Purdom. </span><span class="title">MethCP: Differentially Methylated Region Detection with Change Point Models</span></div>
</li>
<li>
<div class="paper"><span class="authors">Brian Hie, Hyunghoon Cho, Benjamin DeMeo, Bryan Bryson and Bonnie Berger. </span><span class="title">Geometric sketching of single-cell data preserves transcriptional structure</span></div>
</li>
<li>
<div class="paper"><span class="authors">Chirag Jain, Haowen Zhang, Yu Gao and Srinivas Aluru. </span><span class="title">On the Complexity of Sequence to Graph Alignment</span></div>
</li>
<li>
<div class="paper"><span class="authors">Jonathan Jou, Graham Holt, Anna Lowegard and Bruce Donald. </span><span class="title">Minimization-Aware Recursive K* (MARK*): A Novel, Provable Algorithm that Accelerates Ensemble-based Protein Design and Provably Approximates the Energy Landscape</span></div>
</li>
<li>
<div class="paper"><span class="authors">Mikhail Karasikov, Harun Mustafa, Amir Joudaki, Sara Javadzadeh No, Gunnar Rätsch and André Kahles. </span><span class="title">Sparse Binary Relation Representations for Genome Graph Annotation</span></div>
</li>
<li>
<div class="paper"><span class="authors">Younhun Kim, Frederic Koehler, Ankur Moitra, Elchanan Mossel and Govind Ramnarayan. </span><span class="title">How Many Subpopulations is Too Many? Exponential Lower Bounds for Inferring Population Histories</span></div>
</li>
<li>
<div class="paper"><span class="authors">Can Kockan, Kaiyuan Zhu, Natnatee Dokmai, Nikolai Karpov, Oguzhan Kulekci, David Woodruff and Cenk Sahinalp. </span><span class="title">Sketching Algorithms for Genomic Data Analysis and Querying in a Secure Enclave</span></div>
</li>
<li>
<div class="paper"><span class="authors">Alan Kuhnle, Taher Mun, Christina Boucher, Travis Gagie, Ben Langmead and Giovanni Manzini. </span><span class="title">Efficient Construction of a Complete Index for Pan-Genomics Read Alignment</span></div>
</li>
<li>
<div class="paper"><span class="authors">Haoyun Lei, Bochuan Lyu, E. Michael Gertz, Alejandro A. Schaffer, Xulian Shi, Kui Wu, Guibo Li, Liquin Xu, Yong Hu, Michael Dean and Russell Schwartz. </span><span class="title">Tumor Copy Number Deconvolution Integrating Bulk and Single-Cell Sequencing Data</span></div>
</li>
<li>
<div class="paper"><span class="authors">Yunan Luo, Jianzhu Ma, Xiaoming Zhao, Yufeng Su, Yang Liu, Trey Ideker and Jian Peng. </span><span class="title">Mitigating Data Scarcity in Protein Binding Prediction Using Meta-Learning</span></div>
</li>
<li>
<div class="paper"><span class="authors">Joel Mefford, Danny Park, Zhili Zheng, Arthur Ko, Mika Ala-Korpela, Markku Laakso, Paivi Pajukanta, Jian Yang, John Witte and Noah Zaitlen. </span><span class="title">Efficient estimation and applications of cross-validated genetic predictions</span></div>
</li>
<li>
<div class="paper"><span class="authors">Matthew Myers, Gryte Satas and Benjamin Raphael. </span><span class="title">Inferring tumor evolution from longitudinal samples</span></div>
</li>
<li>
<div class="paper"><span class="authors">Weihua Pan, Tao Jiang and Stefano Lonardi. </span><span class="title">OMGS: Optical Map-based Genome Scaffolding</span></div>
</li>
<li>
<div class="paper"><span class="authors">Ali Pazokitoroudi, Yue Wu, Kathryn S. Burch, Kangcheng Hou, Bogdan Pasaniuc and Sriram Sankararaman. </span><span class="title">Scalable multi-component linear mixed models with application to SNP heritability estimation</span></div>
</li>
<li>
<div class="paper"><span class="authors">Leonardo Pellegrina, Cinzia Pizzi and Fabio Vandin. </span><span class="title">Fast Approximation of Frequent k-mers and Applications to Metagenomics</span></div>
</li>
<li>
<div class="paper"><span class="authors">Kristoffer Sahlin and Paul Medvedev. </span><span class="title">De novo clustering of long-read transcriptome data using a greedy, quality-value based algorithm</span></div>
</li>
<li>
<div class="paper"><span class="authors">Shahab Sarmashghi and Vineet Bafna. </span><span class="title">A Note on Computing Interval Overlap Statistics</span></div>
</li>
<li>
<div class="paper"><span class="authors">Itay Sason, Damian Wojtowicz, Welles Robinson, Mark Leiserson, Teresa Przytycka and Roded Sharan. </span><span class="title">A sticky multinomial mixture model of strand-coordinated mutational processes in cancer</span></div>
</li>
<li>
<div class="paper"><span class="authors">Mike Thompson, Zeyuan Johnson Chen, Elior Rahmani and Eran Halperin. </span><span class="title">Recovery of cell-type composition in methylation data using canonical correlation analysis</span></div>
</li>
<li>
<div class="paper"><span class="authors">Sheng Wang, Emily Flynn and Russ Altman. </span><span class="title">GRep: Gene Set Representation via Gaussian Embedding</span></div>
</li>
<li>
<div class="paper"><span class="authors">Yijie Wang, Jan Hoinka and Teresa M. Przytycka. </span><span class="title">Accurate sub-population detection and mapping across single cell experiments with PopCorn</span></div>
</li>
<li>
<div class="paper"><span class="authors">Ziheng Wang, Grace Ht Yeo, Richard Sherwood and David Gifford. </span><span class="title">Disentangled Representations of Cellular Identity</span></div>
</li>
<li>
<div class="paper"><span class="authors">Ye Wu, Ruibang Luo, Henry C.M. Leung, Hing-Fung Ting and Tak-Wah Lam. </span><span class="title">RENET: A Deep Learning Approach for Extracting Gene-Disease Associations from Literature</span></div>
</li>
<li>
<div class="paper"><span class="authors">Yue Wu, Anna Yaschenko, Mohammadreza Hajy Heydary, and Sriram Sankararaman. </span><span class="title">Fast estimation of genetic correlation for Biobank-scale data</span></div>
</li>
<li>
<div class="paper"><span class="authors">Jinbo Xu. <span class="title">Distance-based Protein Folding Powered by Deep Learning</span></div>
</li>
<li>
<div class="paper"><span class="authors">Yang Yang, Yang Zhang, Bing Ren, Jesse Dixon and Jian Ma. </span><span class="title">Comparing 3D Genome Organization in Multiple Species using Phylo-HMRF</span></div>
</li>
<li>
<div class="paper"><span class="authors">Jesse Zhang, Govinda Kamath and David Tse. </span><span class="title">Towards a post-clustering test for differential expression</span></div>
</li>
<li>
<div class="paper"><span class="authors">Martin Zhang, Fei Xia and James Zou. </span><span class="title">AdaFDR: a Fast, Powerful and Covariate-Adaptive Approach for Multiple Hypothesis Testing</span></div>
</li>
</ol>
<p><center></p>
<h2 id="High">Highlight Talks</h2>
<p></center></p>
<ol>
<li>
<div class="paper"><span class="authors">Daifeng Wang, Shuang Liu, Jonathan Warrell, Hyejung Won, Xu Shi, Fabio Navarro, Declan Clarke, Mengting Gu, Prashant Emani, Daniel Geschwind, James Knowles and Mark Gerstein.</span><span class="title"><i>Interpretable deep learning modeling embedding gene regulatory network for understanding functional genomics in neuropsychiatric disorders</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Rachel M. Sherman, Juliet Forman, Valentin Antonescu, Daniela Puiu, Michelle Daya, Nicholas Rafaels, Meher Preethi Boorgula, Sameer Chavan, Candelaria Vergara, Victor E. Ortega, Albert M. Levin, Celeste Eng, Maria Yazdanbakhsh, James G. Wilson, Javier Marrugo, Leslie A. Lange, L. Keoki Williams, Harold Watson, Lorraine B. Ware, Christopher O. Olopade, Olufunmilayo Olopade, Ricardo R. Oliveira, Carole Ober, Dan L. Nicolae, Deborah A. Meyers, Alvaro Mayorga, Jennifer Knight-Madden, Tina Hartert, Nadia N. Hansel, Marilyn G. Foreman, Jean G. Ford, Mezbah U. Faruque, Georgia M. Dunston, Luis Caraballo, Esteban G. Burchard, Eugene R. Bleecker, Maria I. Araujo, Edwin F. Herrera-Paz, Monica Campbell Cassandra Foster, Margaret A. Taub, Terri H. Beaty, Ingo Ruczinski, Rasika A. Mathias, Kathleen C. Barnes and Steven L. Salzberg.</span><span class="title"><i>Assembly of a pan-genome from deep sequencing of 910 humans of African descent</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Tom Aharon Hait, David Amar, Ron Shamir and Ran Elkon.</span><span class="title"><i>FOCS: a novel method for analyzing enhancer and gene activity patterns infers an extensive enhancer–promoter map</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Jack Kuipers, Thomas Thurnherr, Giusi Moffa, Polina Suter, Jonas Behr, Ryan Goosen, Gerhard Christofori and Niko Beerenwinkel.</span><span class="title"><i>Mutational interactions define novel cancer subgroups</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Yaniv Erlich, Tal Shor, Itsik Pe&#8217;Er and Shai Carmi.</span><span class="title"><i>Identity inference of genomic data using long-range familial searches</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Brian Hie, Hyunghoon Cho and Bonnie Berger.</span><span class="title"><i>Realizing private and practical pharmacological collaboration</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Ngoc-Hieu Tran, Rui Qiao, Lei Xin, Xin Chen, Chuyi Liu, Xianglilan Zhang, Baozhen Shan, Ali Ghodsi and Ming Li.</span><span class="title"><i>Deep learning enables de novo peptide sequencing from data-independent-acquisition mass spectrometry</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">David Danko, Dmitrii Meleshko, Daniela Bezdan, Christopher Mason and Iman Hajirasouliha.</span><span class="title"><i>Minerva: An Alignment and Reference Free Approach to Deconvolve Linked-Reads for Metagenomics</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Ali Sinan Köksal, Kirsten Beck, Dylan R Cronin, Aaron McKenna, Nathan D Camp, Saurabh Srivastava, Matthew E MacGilvray, Rastislav Bodík, Alejandro Wolf-Yadlin, Ernest Fraenkel, Jasmin Fisher and Anthony Gitter.</span><span class="title"><i>Synthesizing Signaling Pathways from Temporal Phosphoproteomic Data</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Mahsa Ghanbari, Julia Lasserre and Martin Vingron.</span><span class="title"><i>The Distance Precision Matrix: computing networks from non-linear relationships</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Martin Zhang, Fei Xia and James Zou.</span><span class="title"><i>AdaFDR: a Fast, Powerful and Covariate-Adaptive Approach for Multiple Hypothesis Testing</i></span></div>
</li>
</ol>
<p><center></p>
<h2 id="AccPost">Accepted Posters</h2>
<p></center><br />

<table id="tablepress-6" class="tablepress tablepress-id-6">
<thead>
<tr class="row-1 odd">
	<th class="column-1">ID</th><th class="column-2">Title</th><th class="column-3">Authors</th>
</tr>
</thead>
<tbody class="row-hover">
<tr class="row-2 even">
	<td class="column-1">1</td><td class="column-2">Accelerating sequence alignment to graphs</td><td class="column-3">Chirag Jain, Sanchit Misra, Haowen Zhang, Alexander Dilthey and Srinivas Aluru</td>
</tr>
<tr class="row-3 odd">
	<td class="column-1">2</td><td class="column-2">Scalable multiple whole-genome alignment and locally collinearblock construction with SibeliaZ</td><td class="column-3">Ilia Minkin and Paul Medvedev</td>
</tr>
<tr class="row-4 even">
	<td class="column-1">3</td><td class="column-2">Locality sensitive hashing for the edit distance</td><td class="column-3">Guillaume Marcais, Dan DeBlasio, Prashant Pandey and Carl Kingsford</td>
</tr>
<tr class="row-5 odd">
	<td class="column-1">5</td><td class="column-2">A Network Theory Approach to Identifying Synthetic Lethal Combinations of Pre-Existing Therapeutics in Chemoresistant Cancer Contexts</td><td class="column-3">Jonathon Gast, Matthew Bartolowits, Tony Pedley, Sridar Chittur and Vincent Jo Davisson</td>
</tr>
<tr class="row-6 even">
	<td class="column-1">6</td><td class="column-2">Reconstructing haplotype-specific cancer genome karyotypes with multiple sequencing technologies</td><td class="column-3">Sergey Aganezov, Fritz Sedlazeck, Sara Goodwin, Gayatri Arun, Isac Lee, Sam Kovaka, Michael Kirsche, Rachel Sherman, Robert Wappel, Melissa Kramer, Karen Kostroff, David Spector, Winston Timp, Michael Schatz and Richard McCombie</td>
</tr>
<tr class="row-7 odd">
	<td class="column-1">7</td><td class="column-2">Identifying Retroviral Remnants that Drive an Immune Response to Cancer</td><td class="column-3">James I McDonald, Alejandro Velasquez, Xiaoyun Xing, Michael J Topper, Stephen B Baylin, Ting Wang and Katherine B Chiappinelli</td>
</tr>
<tr class="row-8 even">
	<td class="column-1">8</td><td class="column-2">Investigating how DNA Methylation Shapes the Immune Response to Epigenetic Cancer Therapies</td><td class="column-3">Alejandro Velasquez, James McDonald and Katherine Chiappinelli</td>
</tr>
<tr class="row-9 odd">
	<td class="column-1">9</td><td class="column-2">pathCHEMO: Uncovering (epi) genomic pathways of chemoresistance in lung adenocarcinoma</td><td class="column-3">Nusrat Epsi, Antonina Mitrofanova, Sukanya Panja and Sharon Pine</td>
</tr>
<tr class="row-10 even">
	<td class="column-1">10</td><td class="column-2">An Integer Programing Model for Identification of Cancer-Responsive Subnetwork</td><td class="column-3">Bayarbaatar Amgalan and Makhgal Ganbold</td>
</tr>
<tr class="row-11 odd">
	<td class="column-1">11</td><td class="column-2">Neoantigen Clonal Balance Predicts Immunotherapy Outcomes and Prognosis</td><td class="column-3">Tianshi Lu</td>
</tr>
<tr class="row-12 even">
	<td class="column-1">12</td><td class="column-2">The Clinical Implications of Subclonal Copy Number Alterations in Chronic Lymphocytic Leukemia</td><td class="column-3">Mark Zucker and Kevin Coombes</td>
</tr>
<tr class="row-13 odd">
	<td class="column-1">13</td><td class="column-2">Heterogeneity in refractory acute myeloid leukemia</td><td class="column-3">Sachi Horibata, Gege Gui, Justin Lack, Christin DeStefano, Michael Gottesman and Christopher Hourigan</td>
</tr>
<tr class="row-14 even">
	<td class="column-1">14</td><td class="column-2">Comparing Regulatory Networks between Breast Cancer Cell Line MCF-7 and Human Breast Cancer Tissues</td><td class="column-3">Vy Tran, Alexandra Maertens and Thomas Hartung</td>
</tr>
<tr class="row-15 odd">
	<td class="column-1">15</td><td class="column-2">Genome-wide alteration of CTCF bindings in multiple cancers</td><td class="column-3">Zhenjia Wang, Celestia Fang, Panagiotis Ntziachristos and Chongzhi Zang</td>
</tr>
<tr class="row-16 even">
	<td class="column-1">16</td><td class="column-2">Target gene prediction of a frequently mutated regulatory element in epithelial ovarian cancer</td><td class="column-3">Rosario Corona, Ji-Heui Seo, Xianzhi Lin, Dennis Hazelett, Paulette Mhawech-Fauceglia, Jenny Lester, Sohrab Shah, David Huntsman, Alexander Gusev, Beth Karlan, Benjamin Berman, Matthew Freedman, Simon Gayther and Kate Lawrenson</td>
</tr>
<tr class="row-17 odd">
	<td class="column-1">17</td><td class="column-2">Copy number phylogenetics for single cells</td><td class="column-3">Jack Kuipers, Mustafa Tuncel, Katharina Jahn and Niko Beerenwinkel</td>
</tr>
<tr class="row-18 even">
	<td class="column-1">18</td><td class="column-2">Heterogeneous Imaging Genetics Data Classification via Strictly Orthogonal Nonnegative Matrix Tri-Factorization for Early Detecting Alzheimer's Disease</td><td class="column-3">Kai Liu, Lodewijk Brand, Matthew Miller and Hua Wang</td>
</tr>
<tr class="row-19 odd">
	<td class="column-1">19</td><td class="column-2">Convergence of healthcare records and genetic evidence links cardiovascular dysfunction and Alzheimer’s disease</td><td class="column-3">Hyojung Paik, Junehawk Lee, Chan-Seok Jeong, Jun Sung Park, Jeong Ho Lee, Nadav Rappoport, Atul Butte and Seong Beom Cho</td>
</tr>
<tr class="row-20 even">
	<td class="column-1">20</td><td class="column-2">Assessing prognostic and diagnostic features of (epi)genomics data for neuroblastoma</td><td class="column-3">Katarzyna Wreczycka, Joern Toedling, Falk Hertwig, Johannes Schulte and Altuna Akalin</td>
</tr>
<tr class="row-21 odd">
	<td class="column-1">21</td><td class="column-2">Cross-study ensembling for epigenomic prediction of cardiovascular risk</td><td class="column-3">Kenneth Westerman and José M. Ordovás</td>
</tr>
<tr class="row-22 even">
	<td class="column-1">22</td><td class="column-2">Pathway-based meta-analysis for partially paired transcriptomics analysis in psoriasis</td><td class="column-3">Wing Tung Fung, Joseph T. Wu, Wai Man Mandy Chan, Henry H. Chan and Herbert Pang</td>
</tr>
<tr class="row-23 odd">
	<td class="column-1">23</td><td class="column-2">Predicting Disease Progression of Colorectal Cancer via Unsupervised Learning Methods </td><td class="column-3">Zorian Thornton and Allison Tegge</td>
</tr>
<tr class="row-24 even">
	<td class="column-1">25</td><td class="column-2">Identifying Transcription Factors as Drug Targets for Prostate Carcinoma Considering Protein Moonlighting Behavior</td><td class="column-3">Ashmita Dey, Sagnik Sen, Ramkrishna Mitra and Ujjwal Maulik</td>
</tr>
<tr class="row-25 odd">
	<td class="column-1">26</td><td class="column-2">A Genome-wide Landscape of the Functional Basis of Complex Traits and Diseases</td><td class="column-3">Alexander McKim and Arjun Krishnan</td>
</tr>
<tr class="row-26 even">
	<td class="column-1">27</td><td class="column-2">Predicting ALZ-associated protein biomarkers from multiple evidence sources</td><td class="column-3">Anna Yannakopoulos, Alison Bernstein, Irving Vega and Arjun Krishnan</td>
</tr>
<tr class="row-27 odd">
	<td class="column-1">28</td><td class="column-2">MetaPGN: a pipeline for construction and graphical visualization of annotated pangenome networks</td><td class="column-3">Junhua Li</td>
</tr>
<tr class="row-28 even">
	<td class="column-1">29</td><td class="column-2">Web-based Database System to Detect Fluctuating Selection in a Microbial Mutualism</td><td class="column-3">Hoi Yan Wu, Conard Faraon, Lisa Kim, Sheng Lin, Wai Kwan Shum, Yee Mey Seah, Kristina Hillesland and Wooyoung Kim</td>
</tr>
<tr class="row-29 odd">
	<td class="column-1">31</td><td class="column-2">Surveying bacteriophages of the human gut</td><td class="column-3">Balázs Ligeti, Ágnes Kata Hőnich, Janos Juhasz and Sándor Pongor</td>
</tr>
<tr class="row-30 even">
	<td class="column-1">32</td><td class="column-2">Fast Approximation of Frequent k-mers and Applications to Metagenomics</td><td class="column-3">Leonardo Pellegrina, Cinzia Pizzi and Fabio Vandin</td>
</tr>
<tr class="row-31 odd">
	<td class="column-1">33</td><td class="column-2">MetaCompass: Reference-guided Assembly of Metagenomes</td><td class="column-3">Victoria Cepeda-Espinoza, Bo Liu, Mathieu Almeida, Christopher Hill, Sergey Koren, Todd Treangen and Mihai Pop</td>
</tr>
<tr class="row-32 even">
	<td class="column-1">34</td><td class="column-2">Comparing State-Of-The-Art Denoising Algorithms and Traditional Clustering Algorithms in Microbiome Analysis</td><td class="column-3">Rohan Patil</td>
</tr>
<tr class="row-33 odd">
	<td class="column-1">35</td><td class="column-2">Simultaneous metatranscriptomic and host transcriptional profiling to elucidate signatures of host-pathogen-microbiome interactions</td><td class="column-3">Hayden Brochu, Catherine O'Connell, Jenna Girardi, Erin Harrell, Aiden Jones, Toni Darville, Arlene Seña and Xinxia Peng</td>
</tr>
<tr class="row-34 even">
	<td class="column-1">36</td><td class="column-2">Cedar: scalable, accurate and fast metagenomic abundance estimation</td><td class="column-3">Fatemeh Almodaresi, Mohsen Zakeri and Rob Patro</td>
</tr>
<tr class="row-35 odd">
	<td class="column-1">38</td><td class="column-2">Genes with high network connectivity are enriched for disease heritability</td><td class="column-3">Samuel Kim, Chengzhen Dai, Farhad Hormozdiari, Bryce van de Geijn, Steven Gazal, Yongjin Park, Luke O’connor, Tiffany Amariuta, Po-Ru Loh, Hilary Finucane, Soumya Raychaudhuri and Alkes Price</td>
</tr>
<tr class="row-36 even">
	<td class="column-1">39</td><td class="column-2">Identifying Phenotype-Associated Subnetworks</td><td class="column-3">Yoo-Ah Kim, Rebecca Sarto Basso, Damian Wójtowicz, Dorit S. Hochbaum, Fabio Vandin and Teresa Przytycka</td>
</tr>
<tr class="row-37 odd">
	<td class="column-1">40</td><td class="column-2">Network Methods for Identifying Regulators of Influenza A Virus Infection</td><td class="column-3">Emily Ackerman and Jason Shoemaker</td>
</tr>
<tr class="row-38 even">
	<td class="column-1">41</td><td class="column-2">Classification in biological networks with hypergraphlet kernels</td><td class="column-3">Jose Lugo-Martinez and Predrag Radivojac</td>
</tr>
<tr class="row-39 odd">
	<td class="column-1">42</td><td class="column-2">TreeFix-TP: Phylogenetic Error-Correction for Infectious Disease Transmission Network Inference</td><td class="column-3">Samuel Sledzieski, Chengchen Zhang, Ion Mandoiu and Mukul Bansal</td>
</tr>
<tr class="row-40 even">
	<td class="column-1">43</td><td class="column-2">Building Robust Gene Co-expression Networks from RNA-seq Data</td><td class="column-3">Kayla Johnson and Arjun Krishnan</td>
</tr>
<tr class="row-41 odd">
	<td class="column-1">44</td><td class="column-2">Integrative Network Analysis Discovers Causal RegulatoryNetwork of System Level Measurements</td><td class="column-3">Dongze He, Arda Durmaz, Jean-Eudes Dazard and Gurkan Bebek</td>
</tr>
<tr class="row-42 even">
	<td class="column-1">45</td><td class="column-2">Locating ligand binding sites in G-protein coupled receptors using combined information from docking and sequence conservation</td><td class="column-3">Ashley Ryan Vidad, Stephen Macaspac and Ho Leung Ng</td>
</tr>
<tr class="row-43 odd">
	<td class="column-1">47</td><td class="column-2">PhoglyStruct: Prediction of phosphoglycerylated lysine residues using structural properties of amino acids</td><td class="column-3">Abel Chandra, Alok Sharma, Abdollah Dehzangi, Shoba Ranganathan, Anjeela Jokhan, Kuo-Chen Chou and Tatsuhiko Tsunoda</td>
</tr>
<tr class="row-44 even">
	<td class="column-1">50</td><td class="column-2">De-novo predictions of binding positions within protein domains</td><td class="column-3">Anat Etzion-Fuchs, David A. Todd and Mona Singh</td>
</tr>
<tr class="row-45 odd">
	<td class="column-1">51</td><td class="column-2">Extending protein-DNA binding models to a 16-letter DNA alphabet</td><td class="column-3">Harshit Sahay, Ariel Afek and Raluca Gordan</td>
</tr>
<tr class="row-46 even">
	<td class="column-1">52</td><td class="column-2">Classification of neurotoxic peptides in spider venom using convolutional neural network</td><td class="column-3">Byungjo Lee, In-Wook Hwang, Seung-Tae Kim, Jung-Suk Sung and Wonhee Jang</td>
</tr>
<tr class="row-47 odd">
	<td class="column-1">53</td><td class="column-2">Neural network-based approach for detecting G-quadruplexes</td><td class="column-3">Mira Barshai, Desiree Tillo, Nima Assad, Sreejana Ray, Charles Vinson and Yaron Orenstein</td>
</tr>
<tr class="row-48 even">
	<td class="column-1">54</td><td class="column-2">Comparative Assessment of 6 Docking Software on an Exclusive Metalloprotein-ligand Benchmark with Metal Ions Closely Located at the Binding Site</td><td class="column-3">Süleyman Selim Çınaroğlu and Emel Timuçin</td>
</tr>
<tr class="row-49 odd">
	<td class="column-1">55</td><td class="column-2">Exploratory data analysis and interactive visualization of FTICR-MS data</td><td class="column-3">Lee Ann McCue, Lisa Bramer, Amanda White, Kelly Stratton, Allison Thompson, Daniel Claborne and Kirsten Hofmockel</td>
</tr>
<tr class="row-50 even">
	<td class="column-1">58</td><td class="column-2">Using Clustering Algorithms to Merge Multi-Sample Histone Modification Regions</td><td class="column-3">Daniela Perry and Bogdan Pasaniuc</td>
</tr>
<tr class="row-51 odd">
	<td class="column-1">59</td><td class="column-2">Exploring the Role of Ribosomal Gene Repeats in the Context of Regeneration</td><td class="column-3">Sofia Barreira and Andreas Baxevanis</td>
</tr>
<tr class="row-52 even">
	<td class="column-1">60</td><td class="column-2">Computational Enhancer Prediction: Evaluation and Improvement</td><td class="column-3">Hasiba Asma and Marc S. Halfon</td>
</tr>
<tr class="row-53 odd">
	<td class="column-1">61</td><td class="column-2">HebbPlot: an intelligent tool for learning and visualizing chromatin mark signatures</td><td class="column-3">Hani Z Girgis, Alfredo Velasco and Zachary Reyes</td>
</tr>
<tr class="row-54 even">
	<td class="column-1">62</td><td class="column-2">Reconstructing regulatory networks from multi-omics data using prior information</td><td class="column-3">Johann Hawe, Benjamin Lehne, Melanie Waldenberger, Christian Gieger, John Chambers and Matthias Heinig</td>
</tr>
<tr class="row-55 odd">
	<td class="column-1">63</td><td class="column-2">Analysis of the structural variability of topologically associated domains as revealed by Hi-C</td><td class="column-3">Natalie Sauerwald, Akshat Singhal and Carl Kingsford</td>
</tr>
<tr class="row-56 even">
	<td class="column-1">64</td><td class="column-2">Method to characterize how transcription factors influence development integrating RNA and ChIP-seq</td><td class="column-3">Ashley Conard, Erica Larschan and Charles Lawrence</td>
</tr>
<tr class="row-57 odd">
	<td class="column-1">65</td><td class="column-2">Comparing chromatin accessibility profiles with maximal cliques in temporal graphs</td><td class="column-3">Coby Viner and Michael M. Hoffman</td>
</tr>
<tr class="row-58 even">
	<td class="column-1">66</td><td class="column-2">GGmend: A Mendelian randomization method for finding gene-on-gene regulatory effects in the presence of unobserved confounders</td><td class="column-3">Junghyun Jung, Robert Brown, Wonhee Jang, Bogdan Pasaniuc, Jong Wha Joo and Eleazar Eskin</td>
</tr>
<tr class="row-59 odd">
	<td class="column-1">67</td><td class="column-2">Integrated Probe Design and Quality Control for Capture Hi-C Experiments</td><td class="column-3">Peter Hansen, Salaheddine Ali, Hannah Blau, Daniel Danis, Jochen Hecht, Uwe Kornak, Darío Lupiáñez, Stefan Mundlos, Robin Steinhaus and Peter N. Robinson</td>
</tr>
<tr class="row-60 even">
	<td class="column-1">68</td><td class="column-2">Inferring causal bayesian networks of epigenetic regulation</td><td class="column-3">Ashwini Kumar Sharma</td>
</tr>
<tr class="row-61 odd">
	<td class="column-1">70</td><td class="column-2">Multivariate state space model integrating high precision epigenomic data to elucidate genome-wide chromatin occupancy</td><td class="column-3">Sneha Mitra, Jianling Zhong and Alexander Hartemink</td>
</tr>
<tr class="row-62 even">
	<td class="column-1">71</td><td class="column-2">Integrating MNase-seq and RNA-seq Time Series Data to Study Dynamic Chromatin and Transcriptional Regulation Under Cadmium Stress</td><td class="column-3">Trung Tran, Vinay Tripuraneni, Heather MacAlpine, Sneha Mitra, Yulong Li, Greg Crawford, David MacAlpine and Alexander Hartemink</td>
</tr>
<tr class="row-63 odd">
	<td class="column-1">72</td><td class="column-2">Evolutionary analysis of sequence divergence of cis-regulatory modules associated with changes in chromatin and gene expression in brain tissue of primates.</td><td class="column-3">Abusaid Shaimardanov and Dmitry Svetlichnyy</td>
</tr>
<tr class="row-64 even">
	<td class="column-1">73</td><td class="column-2">Comparative analysis of HiChIP with HiC and ChIP-seq</td><td class="column-3">Yifan Zhang and Chongzhi Zang</td>
</tr>
<tr class="row-65 odd">
	<td class="column-1">74</td><td class="column-2">BARTweb: a web server for functional transcription factor prediction</td><td class="column-3">Wenjing Ma, Zhenjia Wang, Neal E. Magee, Byoung-Do Kim, Yang Chen and Chongzhi Zang</td>
</tr>
<tr class="row-66 even">
	<td class="column-1">75</td><td class="column-2">Cell-type-specific resolution epigenetics without the need for cell sorting or single-cell biology</td><td class="column-3">Elior Rahmani, Regev Schweiger, Brooke Rhead, Lindsey Criswell, Lisa Barcellos, Eleazar Eskin, Saharon Rosset, Sriram Sankararaman and Eran Halperin</td>
</tr>
<tr class="row-67 odd">
	<td class="column-1">76</td><td class="column-2">SICER 2.0: A ChIP-seq broad peak identification tool redesigned</td><td class="column-3">Jin Yong Yoo, Yiren Wang and Chongzhi Zang</td>
</tr>
<tr class="row-68 even">
	<td class="column-1">77</td><td class="column-2">Modeling and correcting intrinsic biases in DNase-seq and ATAC-seq data</td><td class="column-3">Shengen Hu, Lin Liu, Wenjing Ma, Clifford Meyer, Michael Guertin, Tingting Zhang and Chongzhi Zang</td>
</tr>
<tr class="row-69 odd">
	<td class="column-1">78</td><td class="column-2">A Hidden Markov Model-based approach to reconstructing double minute chromosome amplicons</td><td class="column-3">Ruslan Mardugalliamov, Kamal Al Nasr and Matthew Hayes</td>
</tr>
<tr class="row-70 even">
	<td class="column-1">79</td><td class="column-2">soMaCX: A pathway based somatic genome generation framework</td><td class="column-3">Timothy Becker, Andrew Koob and Dong-Guk Shin</td>
</tr>
<tr class="row-71 odd">
	<td class="column-1">80</td><td class="column-2">A toolkit for accelerating genomic analysis using NGS index formats</td><td class="column-3">John Didion</td>
</tr>
<tr class="row-72 even">
	<td class="column-1">82</td><td class="column-2">MeShClust: an intelligent tool for clustering DNA sequences</td><td class="column-3">Benjamin James, Brian Luczak and Hani Zakaria Girgis</td>
</tr>
<tr class="row-73 odd">
	<td class="column-1">83</td><td class="column-2">A secure analysis of environmental and clinical samples using a hybrid cloud profiler</td><td class="column-3">Rachid Ounit, Chris Mason, Stefano Lonardi and Niamh O'Hara</td>
</tr>
<tr class="row-74 even">
	<td class="column-1">84</td><td class="column-2">In silico identification of novel genetic factors associated with longevity in Drosophila</td><td class="column-3">Bethany Hall, Nadia Chuzhanova, Jonathan Crofts and Yvonne Barnett</td>
</tr>
<tr class="row-75 odd">
	<td class="column-1">85</td><td class="column-2">Jetski: Faster Assemblies with Minimizer-Based Removal of Contained Reads</td><td class="column-3">Michael Kirsche and Michael Schatz</td>
</tr>
<tr class="row-76 even">
	<td class="column-1">86</td><td class="column-2">Generalized intra-sequence k-mer distances as an alignment-free sequence-analysis metric</td><td class="column-3">Benjamin Lee and Paul Gamble</td>
</tr>
<tr class="row-77 odd">
	<td class="column-1">87</td><td class="column-2">Bioinformatics Analyses of Next Generation Sequencing Hepatitis C virus samples.</td><td class="column-3">Myrto Kremyda - Vlachou, Andrew Hayward, Laura Shallcross, Stephane Hue, Dan Frampton, Eleni Nastouli and Deenan Pillay</td>
</tr>
<tr class="row-78 even">
	<td class="column-1">88</td><td class="column-2">LtrDetector: A tool-suite for detecting long terminal repeat retrotransposons de-novo</td><td class="column-3">Joseph D Valencia and Hani Z Girgis</td>
</tr>
<tr class="row-79 odd">
	<td class="column-1">89</td><td class="column-2">Predicting primer off-target rate using a sensitive homology search algorithm and deep neural networks</td><td class="column-3">Chang Xu, Frank Reinecke, Raghavendra Padmanabhan, John DiCarlo and Yexun Wang</td>
</tr>
<tr class="row-80 even">
	<td class="column-1">90</td><td class="column-2">Improved Personal Genome Inference Benefits Downstream Analysis</td><td class="column-3">Taher Mun and Benjamin Langmead</td>
</tr>
<tr class="row-81 odd">
	<td class="column-1">91</td><td class="column-2">Refining orthology determination for whole-genome and whole-transcriptome phylogenetics</td><td class="column-3">Madison H. Hansen and Ward C. Wheeler</td>
</tr>
<tr class="row-82 even">
	<td class="column-1">92</td><td class="column-2">Evidence of transcription in novel human genomic sequences</td><td class="column-3">Rachel Sherman and Steven Salzberg</td>
</tr>
<tr class="row-83 odd">
	<td class="column-1">93</td><td class="column-2">Integrative analysis of pooled CRISPR genetic screens using MAGeCKFlute</td><td class="column-3">Bingin Wang, Mei Wang, Wubing Zhang, Wei Li and Xiaole Liu</td>
</tr>
<tr class="row-84 even">
	<td class="column-1">94</td><td class="column-2">AptaSUITE - A Bioinformatics Framework for the comprehensive Analysis of HT-SELEX Experiments</td><td class="column-3">Jan Hoinka, Rolf Backofen and Teresa Przytycka</td>
</tr>
<tr class="row-85 odd">
	<td class="column-1">95</td><td class="column-2">Credibility of Horizontal Gene Transfer Events</td><td class="column-3">Agnieszka Mykowiecka, Anna Muszewska and Pawel Gorecki</td>
</tr>
<tr class="row-86 even">
	<td class="column-1">96</td><td class="column-2">Transcriptomic Analysis of Notch Signaling Perturbation</td><td class="column-3">Grace Solini and Margaret Saha</td>
</tr>
<tr class="row-87 odd">
	<td class="column-1">97</td><td class="column-2">Single-cell Phylogenetic Inference with Copy-number Supported Mutation Losses</td><td class="column-3">Gryte Satas, Simone Zaccaria, Geoffrey Mon and Ben Raphael</td>
</tr>
<tr class="row-88 even">
	<td class="column-1">98</td><td class="column-2">PEPATAC: A portable, optimized ATAC-seq pipeline</td><td class="column-3">Jason Smith, Nathan Sheffield, Vince Reuter, Ryan Corces, Jin Xu, Yuning Wei and Howard Chang</td>
</tr>
<tr class="row-89 odd">
	<td class="column-1">99</td><td class="column-2">Implementation of a high-throughput NGS pipeline: How Amyris scaled from one molecule to dozens</td><td class="column-3">Reshmi Chowdhury and Amoolya Singh</td>
</tr>
<tr class="row-90 even">
	<td class="column-1">100</td><td class="column-2">Glycosylation Flux Analysis: A flux analysis tool to study protein glycosylation</td><td class="column-3">Rudiyanto Gunawan and Sandro Hutter</td>
</tr>
<tr class="row-91 odd">
	<td class="column-1">101</td><td class="column-2">SKmDB: an integrated database of next generation sequencing information in skeletal muscle</td><td class="column-3">Jie Yuan, Jiajian Zhou, Huating Wang and Hao Sun</td>
</tr>
<tr class="row-92 even">
	<td class="column-1">102</td><td class="column-2">A computational analysis of spatial correlations of neural stem cell divisions in the zebrafish brain</td><td class="column-3">Valerio Lupperger, Prisca Chapouton and Carsten Marr</td>
</tr>
<tr class="row-93 odd">
	<td class="column-1">103</td><td class="column-2">Sub-lethal effects of anthropogenic stressors on the cognitive abilities of bumblebee foragers drive population decline and loss of biodiversity</td><td class="column-3">Kevin Heath</td>
</tr>
<tr class="row-94 even">
	<td class="column-1">104</td><td class="column-2">A Comparative Genomics Study of NYC Legionella pneumophila serogroup 1 using Whole Genome Single Nucleotide Polymorphism (wgSNP) Analysis Pipelines</td><td class="column-3">Jade Wang, Edimarlyn Gonzalez, Scott Hughes and Jennifer Rakeman</td>
</tr>
<tr class="row-95 odd">
	<td class="column-1">105</td><td class="column-2">PopVar: population variant detection with false discovery rate (FDR) control</td><td class="column-3">Zhengyuan Wang and Leo Wang-Kit Cheung</td>
</tr>
<tr class="row-96 even">
	<td class="column-1">106</td><td class="column-2">Leveraging allelic heterogeneity to increase power of association testing</td><td class="column-3">Farhad Hormozdiari, Junghyun Jung, Eleazar Eskin and Jong Wha Joo</td>
</tr>
<tr class="row-97 odd">
	<td class="column-1">107</td><td class="column-2">Coordinate Covariation Analysis (COCOA): Understanding Interindividual Variation in Data with Genomic Coordinates</td><td class="column-3">John Lawson, Jason Smith, Francine Garrett-Bakelman, Stefan Bekiranov and Nathan Sheffield</td>
</tr>
<tr class="row-98 even">
	<td class="column-1">108</td><td class="column-2">Meta-analysis of polymyositis and dermatomyositis microarray data sets</td><td class="column-3">Jaeseung Song, Wonhee Jang, Junghyun Jung, Da Eun Kim and Ju Yeon Hong</td>
</tr>
<tr class="row-99 odd">
	<td class="column-1">109</td><td class="column-2">Modular, part-based control of gene expression response time using protein degradation tags</td><td class="column-3">Ethan Jones, Callan Monette, John Marken, Sejal Dhawan, Theresa Gibney, Christine Li, Wukun Liu, Alyssa Luz-Ricca, Xida Ren, Xingyu Zheng and Margaret Saha</td>
</tr>
<tr class="row-100 even">
	<td class="column-1">110</td><td class="column-2">Nested Tree Probabilistic Graphical Models Characterize Evolutionary and Spatial changes in Neural Cell Types</td><td class="column-3">Michael Kleyman, Jing He, Bilge Esin Ozturk, Leah Byrne, William Stauffer and Andreas Pfenning</td>
</tr>
<tr class="row-101 odd">
	<td class="column-1">111</td><td class="column-2">The PAICE Suite: Using Extended Harmonic Oscillators to Identify and Understand Circadian Rhythms in Large Datasets</td><td class="column-3">Hannah De Los Santos, Emily J. Collins, Catherine Mann, April W. Sagan, Meaghan S. Jankowski, Kristin P. Bennett and Jennifer M. Hurley</td>
</tr>
<tr class="row-102 even">
	<td class="column-1">112</td><td class="column-2">GREIN: An Interactive Web Platform for Re-analyzing GEO RNA-seq Data</td><td class="column-3">Naim Mahi</td>
</tr>
<tr class="row-103 odd">
	<td class="column-1">113</td><td class="column-2">Integrative Transcriptome Analysis of Zebra Finch Brain Tissue Exposed to Physiological Levels of Mercury</td><td class="column-3">Kathryn Murphy, Hannah Paros, Margaret Saha and Eric Bradley</td>
</tr>
<tr class="row-104 even">
	<td class="column-1">114</td><td class="column-2">Accurate estimation of transcriptome-wide differential allelic expression</td><td class="column-3">Asia Mendelevich, Svetlana Vinogradova, Saumya Gupta, Andrey Mironov and Alexander Gimelbrant</td>
</tr>
<tr class="row-105 odd">
	<td class="column-1">115</td><td class="column-2">Detecting anomalies in RNA-seq quantification</td><td class="column-3">Cong Ma and Carl Kingsford</td>
</tr>
<tr class="row-106 even">
	<td class="column-1">116</td><td class="column-2">TEMPO: Temporal Modeling of Pathway Outliers</td><td class="column-3">Christopher Pietras, Faith Ocitti and Donna Slonim</td>
</tr>
<tr class="row-107 odd">
	<td class="column-1">117</td><td class="column-2">Alevin efficiently estimates accurate gene abundances from dscRNA-seq data</td><td class="column-3">Avi Srivastava, Laraib Iqbal Malik, Tom Smith, Ian Sudbery and Robert Patro</td>
</tr>
</tbody>
</table>
<!-- #tablepress-6 from cache --></p>
<p><center></p>
<h2 id="LatePost">Accepted Late Breaking Posters</h2>
<p></center><br />

<table id="tablepress-7" class="tablepress tablepress-id-7">
<thead>
<tr class="row-1 odd">
	<th class="column-1">ID</th><th class="column-2">Title</th><th class="column-3">Authors</th>
</tr>
</thead>
<tbody class="row-hover">
<tr class="row-2 even">
	<td class="column-1">41A</td><td class="column-2">Dynamic interaction network inference from longitudinal microbiome data</td><td class="column-3">Jose Lugo-Martinez, Daniel Ruiz Perez, Giri Narasimhan and Ziv Bar-Joseph</td>
</tr>
<tr class="row-3 odd">
	<td class="column-1">119</td><td class="column-2">Simulation-based comparative evaluation of isoform-level quantification for RNA-Seq data</td><td class="column-3">Dimitra Sarantopoulou, Soumyashant Nayak, Nicholas F. Lahens, Thomas G. Brooks and Gregory R. Grant</td>
</tr>
<tr class="row-4 even">
	<td class="column-1">120</td><td class="column-2">NITECAP: A novel method for the identification of circadian behavior in time-course data</td><td class="column-3">Thomas G. Brooks, Cris W. Lawrence, Soumyashant Nayak, Dimitra Sarantopoulou, Nicholas F. Lahens, Tilo Grosser, Garret A. Fitzgerald and Gregory R. Grant</td>
</tr>
<tr class="row-5 odd">
	<td class="column-1">121</td><td class="column-2">TEQILA - A tool enabling queries for investigation of latent associations</td><td class="column-3">Soumyashant Nayak, Eric McGivney, Thomas Brooks, Garret Fitzgerald and Gregory Grant</td>
</tr>
<tr class="row-6 even">
	<td class="column-1">122</td><td class="column-2">BEERS 2.0: The Next Generation of RNA-Seq Simulator</td><td class="column-3">Nicholas Lahens, Thomas Brooks, Dimitra Sarantopoulou, Soumyashant Nayak, Cris Lawrence, Anand Srinivasan, Jonathan Schug, John Hogenesch and Gregory Grant</td>
</tr>
<tr class="row-7 odd">
	<td class="column-1">123</td><td class="column-2">Adaptive Monte Carlo Multiple Testing via Multi-Armed Bandits</td><td class="column-3">Martin Zhang, James Zou and David Tse</td>
</tr>
<tr class="row-8 even">
	<td class="column-1">124</td><td class="column-2">Hera-T: an efficient and accurate approach for quantifying gene abundances from 10X-Chromium data with high rates of non-exonic reads</td><td class="column-3">Thang Tran, Thao Truong, Hy Vuong and Son Pham</td>
</tr>
<tr class="row-9 odd">
	<td class="column-1">125</td><td class="column-2">A revisit of RSEM generative model and its EM algorithm for quantifying transcript abundances</td><td class="column-3">Hy Vuong, Thao Truong, Thang Tran and Son Pham</td>
</tr>
<tr class="row-10 even">
	<td class="column-1">126</td><td class="column-2">BELLA: Berkeley Efficient Long-Read to Long-Read Aligner and Overlapper</td><td class="column-3">Giulia Guidi, Marquita Ellis, Daniel Rokhsar, Katherine Yelick and Aydın Buluç</td>
</tr>
<tr class="row-11 odd">
	<td class="column-1">127</td><td class="column-2">Unified Cox model based multifactor dimensionality reduction method for gene-gene analysis of the survival phenotype</td><td class="column-3">Seungyeoun Lee, Donghee Son, Yongkang Kim, Wenbao Yu and Taesung Park</td>
</tr>
<tr class="row-12 even">
	<td class="column-1">129</td><td class="column-2">Dissecting the Characteristics and Functions of Transcribed Enhancers</td><td class="column-3">Mario Flores and Ivan Ovcharenko</td>
</tr>
<tr class="row-13 odd">
	<td class="column-1">130</td><td class="column-2">Deep Convolutional Neural Networks for Protein Contact Prediction</td><td class="column-3">Badri Adhikari</td>
</tr>
<tr class="row-14 even">
	<td class="column-1">131</td><td class="column-2">nanotatoR: An annotation tool for Optical Mapping</td><td class="column-3">Surajit Bhattacharya, Hayk Barseghyan, Emmanuèle C. Délot and Eric Vilain</td>
</tr>
<tr class="row-15 odd">
	<td class="column-1">132</td><td class="column-2">Computational modeling of stem cell kinetics identifies plausible hematopoietic lineage hierarchies</td><td class="column-3">Lisa Bast, Michele Kyncl, Robert Oostendorp, Katharina Götze, Carsten Marr and Lynette Henkel</td>
</tr>
<tr class="row-16 even">
	<td class="column-1">133</td><td class="column-2">Decomposing Inexact Flows with Application to RNA Transcript Assembly</td><td class="column-3">Lucia Williams and Brendan Mumey</td>
</tr>
<tr class="row-17 odd">
	<td class="column-1">134</td><td class="column-2">An Optimal Combination of Computational Systems Biology Predictions: the MOCA algorithm</td><td class="column-3">Mehmet Eren Ahsen, Gustavo Stolovitzky and Robert Vogel</td>
</tr>
<tr class="row-18 even">
	<td class="column-1">135</td><td class="column-2">PU-Caller: Sensitive Somatic Variant Calling  Using Positive-Unlabeled Learning</td><td class="column-3">Elham Sherafat and Ion Mandoiu</td>
</tr>
<tr class="row-19 odd">
	<td class="column-1">136</td><td class="column-2">MALVA: genotyping by Mapping-free ALlele detection of known Variants</td><td class="column-3">Giulia Bernardini, Paola Bonizzoni, Luca Denti, Marco Previtali and Alexander Schönhuth</td>
</tr>
<tr class="row-20 even">
	<td class="column-1">137</td><td class="column-2">GenMap: Fast and Exact Computation of Genome Mappability</td><td class="column-3">Christopher Pockrandt, Mai Alzamel, Costas Iliopoulos and Knut Reinert</td>
</tr>
<tr class="row-21 odd">
	<td class="column-1">138</td><td class="column-2">Using Deep Learning to understand the grammar of enhancers.</td><td class="column-3">Sanjarbek Hudaiberdiev and Ivan Ovcharenko</td>
</tr>
<tr class="row-22 even">
	<td class="column-1">139</td><td class="column-2">Aberrant splicing and isoform production of kinase proteins is prevalent in metastatic melanoma</td><td class="column-3">David Holland and Laura Elnitski</td>
</tr>
<tr class="row-23 odd">
	<td class="column-1">140</td><td class="column-2">Biclustering analysis of transcriptome big data identifies cancer suppressing microRNAs</td><td class="column-3">Dougu Nam</td>
</tr>
<tr class="row-24 even">
	<td class="column-1">141</td><td class="column-2">Selene: a PyTorch-based deep learning library for sequence data</td><td class="column-3">Kathleen Chen, Evan Cofer, Jian Zhou and Olga Troyanskaya</td>
</tr>
<tr class="row-25 odd">
	<td class="column-1">142</td><td class="column-2">Identifying sample assignment errors in pedigrees</td><td class="column-3">Marteinn Hardarson, Gunnar Palsson, Bjarni Halldorsson, Gisli Masson, Florian Zink and Kári Stefánsson</td>
</tr>
<tr class="row-26 even">
	<td class="column-1">143</td><td class="column-2">Characterizing mutagenic effects of recombinations through a sequence level genetic map</td><td class="column-3">Bjarni Halldorsson, Gunnar Palsson, Ólafur Andri Stefánsson, Hákon Jónsson, Marteinn Hardarson, Hannes P. Eggertsson, Bjarni Gunnarsson, Asmundur Oddsson, Gisli H. Halldorsson, Florian Zink, Sigurjon Gudjonsson, Michael Frigge, Gudmar Thorleifsson, Asgeir Sigurdsson, Simon Stacey, Patrick Sulem, Gisli Masson, Agnar Helgason, Daniel F. Gudbjartsson, Unnur Thorsteinsdottir and Kári Stefánsson</td>
</tr>
<tr class="row-27 odd">
	<td class="column-1">144</td><td class="column-2">HumanBase: A portal for data-driven predictions of gene expression, function, regulation and interactions</td><td class="column-3">Aaron Wong, Julien Funk, Kevin Yao, Chandra Theesfeld and Olga Troyanskaya</td>
</tr>
<tr class="row-28 even">
	<td class="column-1">145</td><td class="column-2">Sharing DNA-binding information across structurally similar proteins enables accurate specificity determination</td><td class="column-3">Joshua Wetzel and Mona Singh</td>
</tr>
<tr class="row-29 odd">
	<td class="column-1">146</td><td class="column-2">Distance Measures for Tumor Evolutionary Trees</td><td class="column-3">Zach DiNardo, Kiran Tomlinson, Anna Ritz and Layla Oesper</td>
</tr>
<tr class="row-30 even">
	<td class="column-1">147</td><td class="column-2">Modeling Clinical and Molecular Covariates of Mutational Process Activity in Cancer</td><td class="column-3">Welles Robinson, Roded Sharan and Mark Leiserson</td>
</tr>
<tr class="row-31 odd">
	<td class="column-1">148</td><td class="column-2">INFERRING ALLELE-SPECIFIC COPY NUMBERS IN SINGLE-CELL SEQUENCING DATA</td><td class="column-3">Simone Zaccaria and Ben Raphael</td>
</tr>
<tr class="row-32 even">
	<td class="column-1">149</td><td class="column-2">Two classes of enhancer stability: stable enhancers active in development and fragile enhancers contributing to evolutionary adaptation</td><td class="column-3">Shan Li, Evgeny Kvon, Axel Visel, Len Pennacchio and Ivan Ovcharenko</td>
</tr>
<tr class="row-33 odd">
	<td class="column-1">150</td><td class="column-2">The first enhancer in an enhancer chain safeguards subsequent enhancer-promoter contacts from a distance</td><td class="column-3">Wei Song, Roded Sharan and Ivan Ovcharenko</td>
</tr>
<tr class="row-34 even">
	<td class="column-1">151</td><td class="column-2">BitMapperBS: a fast and accurate read aligner for whole-genome bisulfite sequencing</td><td class="column-3">Haoyu Cheng and Yun Xu</td>
</tr>
<tr class="row-35 odd">
	<td class="column-1">152</td><td class="column-2">metaFlye: accurate and fast long-read metagenome assembler</td><td class="column-3">Mikhail Kolmogorov and Pavel Pevzner</td>
</tr>
<tr class="row-36 even">
	<td class="column-1">153</td><td class="column-2">Integration of Mouse and Human Single-cell RNA Sequencing Infers Spatial Cell-type Composition in Human Brains</td><td class="column-3">Travis Johnson, Zachary Abrams, Bryan Helm, Peter Neidecker, Raghu Machiraju, Yan Zhang, Kun Huang and Jie Zhang</td>
</tr>
<tr class="row-37 odd">
	<td class="column-1">154</td><td class="column-2">Validating family pedigrees using shared DNA segments</td><td class="column-3">Ladan Doroud, Luong Nguyen, Shiya Song, Jingwen Pei, Harendra Guturu, Keith Noto, Eurie Hong, Ken Chahine and Catherine Ball</td>
</tr>
<tr class="row-38 even">
	<td class="column-1">155</td><td class="column-2">Simulation study of ancestral genome reconstruction from pedigrees</td><td class="column-3">Jingwen Pei, Shiya Song, Ross Curtis, Ladan Doroud, Julie Granka, Keith Noto, Eurie Hong, Ken Chahine and Catherine Ball</td>
</tr>
<tr class="row-39 odd">
	<td class="column-1">156</td><td class="column-2">Identification of copy number variations and translocations in cancer cells from Hi-C data</td><td class="column-3">Abhijit Chakraborty and Ferhat Ay</td>
</tr>
<tr class="row-40 even">
	<td class="column-1">157</td><td class="column-2">FitHiChIP: Identification of significant chromatin contacts from HiChIP/PLAC-seq data</td><td class="column-3">Sourya Bhattacharyya, Vivek Chandra, Vijay Pandurangan and Ferhat Ay</td>
</tr>
<tr class="row-41 odd">
	<td class="column-1">158</td><td class="column-2">CHIP-seq analysis of multiple sclerosis-related differentially expressed genes in circulating B-cells.</td><td class="column-3">Samira Rahimi Rad and Shima Rahimi Rad</td>
</tr>
<tr class="row-42 even">
	<td class="column-1">159</td><td class="column-2">Computational Analysis of Single-Cell Transcriptomics in Ciona Intestinalis Reveals Formation of Novel Neuronal Cell Types with Hybrid Properties</td><td class="column-3">Prakriti Paul, Michael Levine and Mona Singh</td>
</tr>
<tr class="row-43 odd">
	<td class="column-1">160</td><td class="column-2">Batch Effects Correction with Unknown Subtypes</td><td class="column-3">Xiangyu Luo and Yingying Wei</td>
</tr>
<tr class="row-44 even">
	<td class="column-1">161</td><td class="column-2">Selective neuronal vulnerability in Alzheimer's disease: an integrative network-based analysis</td><td class="column-3">Victoria Yao, Jean-Pierre Roussarie, Olga Troyanskaya and Paul Greengard</td>
</tr>
<tr class="row-45 odd">
	<td class="column-1">162</td><td class="column-2">Tissue-aware framework for unraveling rare and complex diseases using biomedical literature</td><td class="column-3">Ruth Dannenfelser, Ran Zhang, Alicja Tadych, Arjun Krishnan and Olga Troyanskaya</td>
</tr>
<tr class="row-46 even">
	<td class="column-1">163</td><td class="column-2">Understanding microglia functional rewiring and disease association through developmental stage specific networks</td><td class="column-3">Ran Zhang, Jose Ledo, Paul Greengard and Olga Troyanskaya</td>
</tr>
<tr class="row-47 odd">
	<td class="column-1">165</td><td class="column-2">Systematic modeling of pan-cancer drug responses using Bayesian multi-source regression</td><td class="column-3">Mehreen Ali, Tero Aittokallio and Suleiman Ali Khan</td>
</tr>
<tr class="row-48 even">
	<td class="column-1">166</td><td class="column-2">Rotation Equivariant and Invariant Networks for Microscopy Image Analysis</td><td class="column-3">Benjamin Chidester, Tianming Zhou, That-Vinh Ton, Minh N. Do and Jian Ma</td>
</tr>
<tr class="row-49 odd">
	<td class="column-1">167</td><td class="column-2">Bayesian modeling of RNA-Seq gene expression count data</td><td class="column-3">Edin Salkovic and Halima Bensmail</td>
</tr>
</tbody>
</table>
<!-- #tablepress-7 from cache --></p>
							</div>

						</article>  

						
					</div><!-- row -->           

				</div><!-- container -->

				<hr class="bottom-dashed">        

			</div>
			
		</section> 

	</main>


	<script type='text/javascript'>
/* <![CDATA[ */
var wpcf7 = {"apiSettings":{"root":"https:\/\/recomb2019.org\/wp-json\/contact-form-7\/v1","namespace":"contact-form-7\/v1"},"cached":"1"};
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/plugins/contact-form-7/includes/js/scripts.js?ver=5.1.1'></script>
<script type='text/javascript' src='https://www.google.com/recaptcha/api.js?render=6LdXS4AUAAAAAFrqX0iBbs1o5KXYzz9oa82yoWbq&#038;ver=3.0'></script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/plugins.js?ver=6.3.5'></script>
<script type='text/javascript'>
/* <![CDATA[ */
var retro_home_vars = {"speed":"500","pausetime":"4000","autosliding":"slider-auto-sliding","pausehover":"slider-pause-hover","url":"https:\/\/recomb2019.org\/wp-admin\/admin-ajax.php","ref":"171fbca9a1","articlenr":"3","portfolionr":"8"};
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/retro.js?ver=6.3.5'></script>
<script type='text/javascript' src='wp-includes/js/wp-embed.min.js?ver=5.0.10'></script>
<script type='text/javascript' src='wp-content/plugins/tablepress/js/jquery.datatables.min.js?ver=1.9.2'></script>
<script type="text/javascript">
jQuery(document).ready(function($){
$('#tablepress-6').dataTable({"order":[],"orderClasses":false,"stripeClasses":["even","odd"],"pagingType":"simple","lengthMenu":[10,20,25,50,100],"pageLength":20});
$('#tablepress-7').dataTable({"order":[],"orderClasses":false,"stripeClasses":["even","odd"],"pagingType":"simple","lengthMenu":[10,20,25,50,100],"pageLength":20});
});
</script><script type="text/javascript">
( function( grecaptcha, sitekey ) {

	var wpcf7recaptcha = {
		execute: function() {
			grecaptcha.execute(
				sitekey,
				{ action: 'homepage' }
			).then( function( token ) {
				var forms = document.getElementsByTagName( 'form' );

				for ( var i = 0; i < forms.length; i++ ) {
					var fields = forms[ i ].getElementsByTagName( 'input' );

					for ( var j = 0; j < fields.length; j++ ) {
						var field = fields[ j ];

						if ( 'g-recaptcha-response' === field.getAttribute( 'name' ) ) {
							field.setAttribute( 'value', token );
							break;
						}
					}
				}
			} );
		}
	};

	grecaptcha.ready( wpcf7recaptcha.execute );

	document.addEventListener( 'wpcf7submit', wpcf7recaptcha.execute, false );

} )( grecaptcha, '6LdXS4AUAAAAAFrqX0iBbs1o5KXYzz9oa82yoWbq' );
</script>

	</body>
	
</html>
<!--
Performance optimized by W3 Total Cache. Learn more: https://www.w3-edge.com/products/

Object Caching 11/111 objects using apc
Page Caching using apc (SSL caching disabled) 
Database Caching using apc

Served from: recomb2019.org @ 2020-09-27 09:02:19 by W3 Total Cache
-->
