<!DOCTYPE html>

<html class="no-js" lang="en-US">

	<head>

		<meta charset="UTF-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">

		
		
				

		<title>SCHEDULE &#8211; RECOMB 2019</title>
<link rel='dns-prefetch' href='//www.google.com' />
<link rel='dns-prefetch' href='//fonts.googleapis.com' />
<link rel='dns-prefetch' href='//s.w.org' />
<link rel="alternate" type="application/rss+xml" title="RECOMB 2019 &raquo; Feed" href="feed/" />
<link rel="alternate" type="application/rss+xml" title="RECOMB 2019 &raquo; Comments Feed" href="comments/feed/" />
		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11\/72x72\/","ext":".png","svgUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11\/svg\/","svgExt":".svg","source":{"concatemoji":"https:\/\/recomb2019.org\/wp-includes\/js\/wp-emoji-release.min.js?ver=5.0.12"}};
			!function(e,a,t){var n,r,o,i=a.createElement("canvas"),p=i.getContext&&i.getContext("2d");function s(e,t){var a=String.fromCharCode;p.clearRect(0,0,i.width,i.height),p.fillText(a.apply(this,e),0,0);e=i.toDataURL();return p.clearRect(0,0,i.width,i.height),p.fillText(a.apply(this,t),0,0),e===i.toDataURL()}function c(e){var t=a.createElement("script");t.src=e,t.defer=t.type="text/javascript",a.getElementsByTagName("head")[0].appendChild(t)}for(o=Array("flag","emoji"),t.supports={everything:!0,everythingExceptFlag:!0},r=0;r<o.length;r++)t.supports[o[r]]=function(e){if(!p||!p.fillText)return!1;switch(p.textBaseline="top",p.font="600 32px Arial",e){case"flag":return s([55356,56826,55356,56819],[55356,56826,8203,55356,56819])?!1:!s([55356,57332,56128,56423,56128,56418,56128,56421,56128,56430,56128,56423,56128,56447],[55356,57332,8203,56128,56423,8203,56128,56418,8203,56128,56421,8203,56128,56430,8203,56128,56423,8203,56128,56447]);case"emoji":return!s([55358,56760,9792,65039],[55358,56760,8203,9792,65039])}return!1}(o[r]),t.supports.everything=t.supports.everything&&t.supports[o[r]],"flag"!==o[r]&&(t.supports.everythingExceptFlag=t.supports.everythingExceptFlag&&t.supports[o[r]]);t.supports.everythingExceptFlag=t.supports.everythingExceptFlag&&!t.supports.flag,t.DOMReady=!1,t.readyCallback=function(){t.DOMReady=!0},t.supports.everything||(n=function(){t.readyCallback()},a.addEventListener?(a.addEventListener("DOMContentLoaded",n,!1),e.addEventListener("load",n,!1)):(e.attachEvent("onload",n),a.attachEvent("onreadystatechange",function(){"complete"===a.readyState&&t.readyCallback()})),(n=t.source||{}).concatemoji?c(n.concatemoji):n.wpemoji&&n.twemoji&&(c(n.twemoji),c(n.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
<link rel='stylesheet' id='wp-block-library-css'  href='wp-includes/css/dist/block-library/style.min.css?ver=5.0.12' type='text/css' media='all' />
<link rel='stylesheet' id='contact-form-7-css'  href='wp-content/plugins/contact-form-7/includes/css/styles.css?ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='mptt-style-css'  href='wp-content/plugins/mp-timetable/media/css/style.css?ver=2.2.1' type='text/css' media='all' />
<link rel='stylesheet' id='parent-style-css'  href='wp-content/themes/Retro-4/style.css?ver=5.0.12' type='text/css' media='all' />
<link rel='stylesheet' id='child-style-css'  href='wp-content/themes/Retro-4-child/style.css?ver=6.3.5' type='text/css' media='all' />
<link rel='stylesheet' id='retro-fonts-css'  href='https://fonts.googleapis.com/css?family=Roboto+Slab%3A400%2C700%7COswald%3A400%2C700&#038;subset&#038;ver=5.0.12' type='text/css' media='all' />
<link rel='stylesheet' id='retro-icons-css'  href='wp-content/themes/Retro-4/fonts/retro-icons.css?ver=5.0.12' type='text/css' media='all' />
<link rel='stylesheet' id='retro-style-css'  href='wp-content/themes/Retro-4-child/style.css?ver=5.0.12' type='text/css' media='all' />
<link rel='stylesheet' id='tablepress-default-css'  href='wp-content/plugins/tablepress/css/default.min.css?ver=1.9.2' type='text/css' media='all' />
<link rel='stylesheet' id='tmm-css'  href='wp-content/plugins/team-members/inc/css/tmm_style.min.css?ver=5.0.12' type='text/css' media='all' />
<script type='text/javascript' src='wp-includes/js/jquery/jquery.js?ver=1.12.4'></script>
<script type='text/javascript' src='wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1'></script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/modernizr.js?ver=2.7.1'></script>
<link rel='https://api.w.org/' href='wp-json/' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="xmlrpc.php?rsd" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="wp-includes/wlwmanifest.xml" /> 
<meta name="generator" content="WordPress 5.0.12" />
<link rel="canonical" href="schedule/" />
<link rel='shortlink' href='?p=797' />
<link rel="alternate" type="application/json+oembed" href="wp-json/oembed/1.0/embed?url=https%3A%2F%2Frecomb2019.org%2Fschedule%2F" />
<link rel="alternate" type="text/xml+oembed" href="wp-json/oembed/1.0/embed?url=https%3A%2F%2Frecomb2019.org%2Fschedule%2F&#038;format=xml" />
<noscript><style type="text/css">.mptt-shortcode-wrapper .mptt-shortcode-table:first-of-type{display:table!important}.mptt-shortcode-wrapper .mptt-shortcode-table .mptt-event-container:hover{height:auto!important;min-height:100%!important}body.mprm_ie_browser .mptt-shortcode-wrapper .mptt-event-container{height:auto!important}@media (max-width:767px){.mptt-shortcode-wrapper .mptt-shortcode-table:first-of-type{display:none!important}}</style></noscript><style>body, input, textarea, .blog-list .post-title h3, .blog-list h4.post-meta, .portfolio-list ul li h3, .single h2.post-title, .page h2.post-title { font-family: "Roboto Slab", sans-serif; }</style><style>.hentry-content h1, .hentry-content h2, .hentry-content h3, .hentry-content h4, .hentry-content h5, .hentry-content h6, .section-subtitle, .blog-list h4.post-meta, .single h4.post-meta, .widget h3, .comments h3, .comments h4, .comment-author, .more-posts { font-family: "Oswald", serif; }</style><link rel="icon" href="wp-content/uploads/2018/09/RECOMB2019thumbnail-150x150-1-80x80.png" sizes="32x32" />
<link rel="icon" href="wp-content/uploads/2018/09/RECOMB2019thumbnail-150x150-1.png" sizes="192x192" />
<link rel="apple-touch-icon-precomposed" href="wp-content/uploads/2018/09/RECOMB2019thumbnail-150x150-1.png" />
<meta name="msapplication-TileImage" content="wp-content/uploads/2018/09/RECOMB2019thumbnail-150x150-1.png" />
		<style type="text/css" id="wp-custom-css">
			.tablepress-id-7 .column-1 {
	min-width: 30pt;
	width: 8%;
	max-width: 40pt;
}

.tablepress-id-7 .column-2 {
	width: 52%;
}

.tablepress-id-7 .column-3 {
	width: 40%;
}		</style>
	
		<style>

@font-face {
  font-family: "BazarMedium";
  src: url("wp-content/themes/Retro-4/fonts/bazarmedium.eot");
  src: url("wp-content/themes/Retro-4/fonts/bazarmedium.eot?#iefix") format("embedded-opentype"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.woff") format("woff"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.ttf") format("truetype"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.svg#BazarMedium") format("svg");
  font-weight: normal;
  font-style: normal;
}

/* Fix ugly font rendering */
@media screen and (-webkit-min-device-pixel-ratio:0) {
    @font-face {
        font-family: "BazarMedium";
            src: url("wp-content/themes/Retro-4/fonts/bazarmedium.svg#BazarMedium") format("svg");
    }
}

h1, h2, h3, h4, h5, h6,
nav ul#main-nav li {
  font-family: "BazarMedium";
  text-transform: uppercase;
}




	#logo a {

		background-image: url(wp-content/uploads/2018/11/RECOMB_plain-logo-tan.png);

	}






@import url('http://fonts.googleapis.com/css?family=PT+Sans:400,700');

.sticky-note{
  overflow-y: auto;
}
</style>
	</head>

	<body id="start" class="page-template-default page page-id-797 fixed-header left-logo">
<header role="banner">

    <div class="header-inner " style="background-color: #004065">

        <div class="container">

            <div class="row clear">

                
<div id="logo">

	<a href="index.html" title="RECOMB 2019" >

		<h1>RECOMB 2019</h1>

	</a>
	
</div>
                <nav role="navigation">

 <ul id="main-nav" class="clear"><li id="menu-item-975" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="awards.html">AWARDS</a></li>
<li id="menu-item-26" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="keynotes.html">Keynotes</a></li>
<li id="menu-item-800" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="schedule.html">SCHEDULE</a></li>
<li id="menu-item-853" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="materials.html">MATERIALS</a></li>
<li id="menu-item-921" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="venue.html">VENUE</a></li>
<li id="menu-item-28" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="committees.html">COMMITTEES</a></li>
<li id="menu-item-468" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="travel-fellowship.html">Fellowship</a></li>
<li id="menu-item-320" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="sponsors.html">SPONSORS</a></li>
<li id="menu-item-307" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="contact-form.html">Contact Us</a></li>
</ul>

                </nav>

            </div><!-- row -->

        </div><!-- container -->

        <hr class="bottom-dashed">  

    </div><!-- header-inner -->   

</header>
			

	
	<main role="main">

		<section>

			<div class="section-inner section-post " style="background-color: #fffffe">
				
				<hr class="top-dashed"> 

				<div class="container">

					<div class="row clear">

						<article class="col col-12 tablet-full mobile-full">

							<h2 class="post-title">SCHEDULE</h2>

							<hr class="single-hr">

							<div class="hentry-content">
								<p style="text-align: center;"><a href="materials.html">RECOMB 2019 materials is avaliable here.</a></p>
<ul>
<li><a href="#Sat">Saturday, May 4</a></li>
<li><a href="#Sun">Sunday, May 5</a></li>
<li><a href="#Mon">Monday, May 6</a></li>
<li><a href="#Tue">Tuesday, May 7</a></li>
<li><a href="#Wed">Wednesday, May 8</a></li>
</ul>

<table id="tablepress-1" class="tablepress tablepress-id-1">
<thead>
<tr class="row-1">
	<th class="column-1"><center><p style="font-size:22px; display:inline; text-align: center"><a style="color: black" id="Sat"><strong>Saturday, May 4</strong></p></center></th>
</tr>
</thead>
<tbody class="row-hover">
<tr class="row-2">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Registration</strong></p><br>8:00 AM - 6:00 PM<br><strong>Venue:</strong> Marvin Center, Lobby</center></td>
</tr>
</tbody>
</table>


<table id="tablepress-2" class="tablepress tablepress-id-2">
<thead>
<tr class="row-1">
	<th class="column-1"><center><p style="font-size:22px; display:inline;"><a style="color: black" id="Sun"><strong>Sunday, May 5</strong></a></p></center></th>
</tr>
</thead>
<tbody class="row-hover">
<tr class="row-2">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Registration</strong></p><br>8:00 AM - 8:45 AM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-3">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Opening and Welcome Remarks</strong></p><br>8:45 AM - 9:00 AM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-4">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>Keynote</strong></p><br><p style="font-size:20px; display:inline; color: #c8b18b"><strong>The Landscape of Disease Comorbidities</strong></p><br><p style="font-size:16px; display:inline; color: #c8b18b"><a style="color: #c8b18b" href="alfonso-valencia/">Alfonso Valencia</a></p><br />
<p style="display:inline; color: #c8b18b">9:00 AM - 10:00 AM</p><br><p style="display:inline; color: #c8b18b"><strong>Chair:</strong> Martin Vingron (Max Planck Institute for Molecular Genetics)</p></center></td>
</tr>
<tr class="row-5">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>GENETICS AND ASSOCIATION STUDIES</strong></p><br>10:00 AM - 11:00 AM<br><strong>Chair:</strong> Itsik Pe'er<br><br />
<p style="font-size:18px; display:inline;"><strong>Interpretable deep learning modeling embedding gene regulatory network for understanding functional genomics in neuropsychiatric disorders <mark>[Highlight]</mark></strong> </p><br><strong>Authors:</strong> Daifeng Wang, Shuang Liu, Jonathan Warrell, Hyejung Won, Xu Shi, Fabio Navarro, Declan Clarke, Mengting Gu, Prashant Emani, Daniel Geschwind, James Knowles and <span style="background-color:#c8b18b;"><strong>Mark Gerstein</strong></span><br><br />
<p style="font-size:18px; display:inline;"><strong>Scalable multi-component linear mixed models with application to SNP heritability estimation</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Ali Pazokitoroudi</strong></span>, Yue Wu, Kathryn S. Burch, Kangcheng Hou, Bogdan Pasaniuc and Sriram Sankararaman<br><br />
<p style="font-size:18px; display:inline;"><strong>Fast estimation of genetic correlation for Biobank-scale data</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Yue Wu</strong></span>, Anna Yaschenko, Mohammadreza Hajy Heydary, and Sriram Sankararaman</center></td>
</tr>
<tr class="row-6">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Coffee Break</strong></p><br>11:00 AM - 11:20 AM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-7">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>SEQUENCING I</strong></p><br>11:20 AM - 12:00 PM<br><strong>Chair:</strong> Paul Medvedev<br> <br />
<p style="font-size:18px; display:inline;"><strong>Efficient Construction of a Complete Index for Pan-Genomics Read Alignment</strong> </p><br><strong>Authors:</strong>  <span style="background-color:#c8b18b;"><strong>Alan Kuhnle</strong></span>, Taher Mun, Christina Boucher, Travis Gagie, Ben Langmead and Giovanni Manzini<br><br />
<p style="font-size:18px; display:inline;"><strong> Assembly of a pan-genome from deep sequencing of 910 humans of African descent <mark>[Highlight]</mark></strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Rachel M. Sherman</strong></span>, Juliet Forman, Valentin Antonescu, Daniela Puiu, Michelle Daya, Nicholas Rafaels, Meher Preethi Boorgula, Sameer Chavan, Candelaria Vergara, Victor E. Ortega, Albert M. Levin, Celeste Eng, Maria Yazdanbakhsh, James G. Wilson, Javier Marrugo, Leslie A. Lange, L. Keoki Williams, Harold Watson, Lorraine B. Ware, Christopher O. Olopade, Olufunmilayo Olopade, Ricardo R. Oliveira, Carole Ober, Dan L. Nicolae, Deborah A. Meyers, Alvaro Mayorga, Jennifer Knight-Madden, Tina Hartert, Nadia N. Hansel, Marilyn G. Foreman, Jean G. Ford, Mezbah U. Faruque, Georgia M. Dunston, Luis Caraballo, Esteban G. Burchard, Eugene R. Bleecker, Maria I. Araujo, Edwin F. Herrera-Paz, Monica Campbell Cassandra Foster, Margaret A. Taub, Terri H. Beaty, Ingo Ruczinski, Rasika A. Mathias, Kathleen C. Barnes and Steven L. Salzberg </center></td>
</tr>
<tr class="row-8">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Lunch Break</strong></p><br>12:00 PM - 1:30 PM</center></td>
</tr>
<tr class="row-9">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>Keynote</strong></p><br><p style="font-size:20px; display:inline; color: #c8b18b"><strong>Evolutionary mathematical modeling enables optimization of glioblastoma treatment strategies</strong></p><br><p style="font-size:16px; display:inline; color: #c8b18b"><a style="color: #c8b18b" href="franziska-michor/">Franziska Michor</a></p><br />
<p style="display:inline; color: #c8b18b">1:30 PM - 2:30 PM</p><br><p style="display:inline; color: #c8b18b"><strong>Chair:</strong> Teresa Przytycka (NIH)</p></center></td>
</tr>
<tr class="row-10">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>CANCER I</strong></p><br>2:30 PM - 3:10 PM<br><strong>Chair:</strong> Russell Schwartz<br><br />
<p style="font-size:18px; display:inline;"><strong> Inferring tumor evolution from longitudinal samples </strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Matthew Myers</strong></span>, Gryte Satas and Benjamin Raphael<br><br />
<p style="font-size:18px; display:inline;"><strong>A sticky multinomial mixture model of strand-coordinated mutational processes in cancer </strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Ali Itay Sason</strong></span>, Damian Wojtowicz, Welles Robinson, Mark Leiserson, Teresa Przytycka and Roded Sharan</center><br />
</td>
</tr>
<tr class="row-11">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Coffee Break</strong></p><br>3:10 PM - 3:30 PM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-12">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>REGULATORY GENOMICS</strong></p><br>3:30 PM - 4:30 PM<br><strong>Chair:</strong> Manolis Kellis<br><br />
<p style="font-size:18px; display:inline;"><strong> FOCS: a novel method for analyzing enhancer and gene activity patterns infers an extensive enhancer–promoter map  <mark>[Highlight]</mark></strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Tom Aharon Hait</strong></span>, David Amar, Ron Shamir and Ran Elkon<br><br />
<p style="font-size:18px; display:inline;"><strong>MethCP: Differentially Methylated Region Detection with Change Point Models</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Boying Gong</strong></span> and Elizabeth Purdom<br><br />
<p style="font-size:18px; display:inline;"><strong>Distinguishing biological from technical sources of variation by leveraging multiple methylation datasets</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Mike Thompson</strong></span>, Zeyuan Johnson Chen, Elior Rahmani and Eran Halperin</center><br />
</td>
</tr>
<tr class="row-13">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>STATISTICAL  FOUNDATIONS</strong></p><br>4:30 PM - 5:50 PM<br><strong>Chair:</strong> Simon Gravel<br><br />
<p style="font-size:18px; display:inline;"><strong> AdaFDR: a Fast, Powerful and Covariate-Adaptive Approach for Multiple Hypothesis Testing  <mark>[Highlight]</mark></strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Martin Zhang</strong></span>, Fei Xia and James Zou<br><br />
<p style="font-size:18px; display:inline;"><strong>A Note on Computing Interval Overlap Statistics</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Shahab Sarmashghi</strong></span> and Vineet Bafna<br><br />
<p style="font-size:18px; display:inline;"><strong>Efficient estimation and applications of cross-validated genetic predictions</strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Joel Mefford</strong></span>, Danny Park, Zhili Zheng, Arthur Ko, Mika Ala-Korpela, Markku Laakso, Paivi Pajukanta, Jian Yang, John Witte and Noah Zaitlen<br><br />
<p style="font-size:18px; display:inline;"><strong>How Many Subpopulations is Too Many? Exponential Lower Bounds for Inferring Population Histories</strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Younhun Kim</strong></span>, Frederic Koehler, Ankur Moitra, Elchanan Mossel and Govind Ramnarayan</center></td>
</tr>
<tr class="row-14">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>RECOMB Business Meeting</strong></p><br>6:00 PM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
</tbody>
</table>


<table id="tablepress-3" class="tablepress tablepress-id-3">
<thead>
<tr class="row-1">
	<th class="column-1"><center><p style="font-size:22px; display:inline; text-align: center"><a style="color: black" id="Mon"><strong>Monday, May 6</strong></a></p></center></th>
</tr>
</thead>
<tbody class="row-hover">
<tr class="row-2">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>Keynote</strong></p><br><p style="font-size:20px; display:inline; color: #c8b18b"><strong>Harnessing genetic interactions to advance whole-exome precision cancer treatment</strong></p><br><p style="font-size:16px; display:inline; color: #c8b18b"><a style="color: #c8b18b" href="eytan-ruppin/">Eytan Ruppin</a></p><br><p style="display:inline; color: #c8b18b">9:00 AM - 10:00 AM</p><br><p style="display:inline; color: #c8b18b"><strong>Chair:</strong> Cenk Sahinalp (Indiana University)</p><br><p style="display:inline; color: #c8b18b"><strong>Venue:</strong> Lisner Auditorium</p></center></td>
</tr>
<tr class="row-3">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>CANCER II</strong></p><br>10:00 AM - 10:40 AM<br><strong>Chair:</strong> Max Leiserson<br><br />
<p style="font-size:18px; display:inline;"><strong> Mutational interactions define novel cancer subgroups <mark>[Highlight]</mark></strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Jack Kuipers</strong></span>, Thomas Thurnherr, Giusi Moffa, Polina Suter, Jonas Behr, Ryan Goosen, Gerhard Christofori and Niko Beerenwinkel<br><br />
<p style="font-size:18px; display:inline;"><strong>Tumor Copy Number Deconvolution Integrating Bulk and Single-Cell Sequencing Data</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Haoyun Lei</strong></span>, Bochuan Lyu, E. Michael Gertz, Alejandro A. Schaffer, Xulian Shi, Kui Wu, Guibo Li, Liquin Xu, Yong Hu, Michael Dean and Russell Schwartz</center></td>
</tr>
<tr class="row-4">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Coffee Break</strong></p><br>10:40 AM - 11:00 AM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-5">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>SINGLE CELL I</strong></p><br>11:00 AM - 12:00 PM<br><strong>Chair:</strong> Ben Raphael<br><br />
<p style="font-size:18px; display:inline;"><strong>Geometric sketching of single-cell data preserves transcriptional structure</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Brian Hie</strong></span>, Hyunghoon Cho, Benjamin DeMeo, Bryan Bryson and Bonnie Berger<br><br />
<p style="font-size:18px; display:inline;"><strong>Dynamic pseudo-time warping of complex single-cell trajectories</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Van Hoan Do</strong></span>, Mislav Blažević, Pablo Monteagudo, Luka Borozan, Khaled Elbassioni, Soeren Laue, Francisca Rojas Ringeling, Domagoj Matijevic and Stefan Canzar<br><br />
<p style="font-size:18px; display:inline;"><strong>Accurate sub-population detection and mapping across single cell experiments with PopCorn</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Yijie Wang</strong></span>, Jan Hoinka and Teresa M. Przytycka</center></td>
</tr>
<tr class="row-6">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Lunch Break</strong></p><br>12:00 PM - 1:30 PM</center></td>
</tr>
<tr class="row-7">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>Keynote</strong></p><br><p style="font-size:20px; display:inline; color: #c8b18b"><strong>Evolution-CSI: hunting for evidence in the protein universe</strong></p><br><p style="font-size:16px; display:inline; color: #c8b18b"><a style="color: #c8b18b" href="rachel-kolodny/">Rachel Kolodny</a></p><br><p style="display:inline; color: #c8b18b">1:30 PM - 2:30 PM</p><br><p style="display:inline; color: #c8b18b"><strong>Chair:</strong> Lenore Cowen (Tufts University)</p><br><p style="display:inline; color: #c8b18b"><strong>Venue:</strong> Lisner Auditorium</p></center></td>
</tr>
<tr class="row-8">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>PROTEIN STRUCTURE</strong></p><br>2:30 PM - 3:30 PM<br><strong>Chair:</strong> Dario Ghersi<br><br />
<p style="font-size:18px; display:inline;"><strong>Mitigating Data Scarcity in Protein Binding Prediction Using Meta-Learning</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Yunan Luo</strong></span>, Jianzhu Ma, Xiaoming Zhao, Yufeng Su, Yang Liu, Trey Ideker and Jian Peng<br><br />
<p style="font-size:18px; display:inline;"><strong>Distance-based Protein Folding Powered by Deep Learning</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Jinbo Xu</strong></span><br><br />
<p style="font-size:18px; display:inline;"><strong>Minimization-Aware Recursive K* (MARK*): A Novel, Provable Algorithm that Accelerates Ensemble-based Protein Design and Provably Approximates the Energy Landscape </strong> </p><br><strong>Authors:</strong> Jonathan Jou, <span style="background-color:#c8b18b;"><strong>Graham Holt</strong></span>, Anna Lowegard and Bruce Donald</center></td>
</tr>
<tr class="row-9">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Coffee Break</strong></p><br>3:30 PM - 3:50 PM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-10">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>SINGLE CELL II</strong></p><br>3:50 PM - 4:50 PM<br><strong>Chair:</strong> Fabio Vandin<br><br />
<p style="font-size:18px; display:inline;"><strong>netNMF: A network regularization algorithm for dimensionality reduction and imputation of single-cell expression data</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Rebecca Elyanow</strong></span>, Bianca Dumitrascu, Barbara E. Engelhardt, and Benjamin J. Raphael<br><br />
<p style="font-size:18px; display:inline;"><strong>Disentangled Representations of Cellular Identity</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Ziheng Wang</strong></span>, Grace Ht Yeo, Richard Sherwood and David Gifford<br><br />
<p style="font-size:18px; display:inline;"><strong>Towards a post-clustering test for differential expression</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Jesse Zhang</strong></span>, Govinda Kamath and David Tse</center></td>
</tr>
<tr class="row-11">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>TEXT MINING</strong></p><br>4:50 PM - 5:30 PM<br><strong>Chair:</strong> Roded Sharan<br><br />
<p style="font-size:18px; display:inline;"><strong> RENET: A Deep Learning Approach for Extracting Gene-Disease Associations from Literature </strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Ye Wu</strong></span>, Ruibang Luo, Henry C.M. Leung, Hing-Fung Ting and Tak-Wah Lam<br><br />
<p style="font-size:18px; display:inline;"><strong>Identifying clinical terms in free-text notes using ontology-guided machine learning</strong> </p><br><strong>Authors:</strong> Aryan Arbabi, David Adams, Sanja Fidler and <span style="background-color:#c8b18b;"><strong>Michael Brudno</strong></span></center></td>
</tr>
<tr class="row-12">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Break</strong></p><br>5:30 PM - 6:00 PM</center></td>
</tr>
<tr class="row-13">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>Poster Session and Reception</strong></p><br><p style="display:inline; color: #c8b18b">6:00 PM - 8:00 PM</p><br><p style="display:inline; color: #c8b18b"><strong>Venue:</strong> Marvin Center, Grand Ballroom</p></center> </td>
</tr>
</tbody>
</table>


<table id="tablepress-4" class="tablepress tablepress-id-4">
<thead>
<tr class="row-1">
	<th class="column-1"><center><p style="font-size:22px; display:inline; text-align: center"><strong><a style="color: black" id="Tue">Tuesday, May 7</a></strong></p></center></th>
</tr>
</thead>
<tbody class="row-hover">
<tr class="row-2">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>Keynote</strong></p><br><p style="font-size:20px; display:inline; color: #c8b18b"><strong>Innovating in Biomedical Data Science for Precision Health</strong></p><br><p style="font-size:16px; display:inline; color: #c8b18b"><a style="color: #c8b18b" href="carlos-bustamante/">Carlos Bustamante</a></p><br><p style="display:inline; color: #c8b18b">9:00 AM - 10:00 AM</p><br><p style="display:inline; color: #c8b18b"><strong>Chair:</strong> Bonnie Berger (MIT)</p><br><p style="display:inline; color: #c8b18b"><strong>Venue:</strong> Lisner Auditorium</p></center></td>
</tr>
<tr class="row-3">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>GENOMIC PRIVACY</strong></p><br>10:00 AM - 11:00 AM<br><strong>Chair:</strong> Cenk Sahinalp<br><br />
<p style="font-size:18px; display:inline;"><strong>Identity inference of genomic data using long-range familial searches <mark>[Highlight]</mark> </strong> </p><br><strong>Authors:</strong> Yaniv Erlich, Tal Shor, <span style="background-color:#c8b18b;"><strong>Itsik Pe'Er</strong></span> and Shai Carmi<br><br />
<p style="font-size:18px; display:inline;"><strong>Sketching Algorithms for Genomic Data Analysis and Querying in a Secure Enclave</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Can Kockan</strong></span>, <span style="background-color:#c8b18b;"><strong>Kaiyuan Zhu</strong></span>, Natnatee Dokmai, Nikolai Karpov, Oguzhan Kulekci, David Woodruff and Cenk Sahinalp<br><br />
<p style="font-size:18px; display:inline;"><strong>Realizing private and practical pharmacological collaboration <mark>[Highlight]</mark></strong> </p><br><strong>Authors:</strong> Brian Hie, <span style="background-color:#c8b18b;"><strong>Hyunghoon Cho</strong></span> and Bonnie Berger</center></td>
</tr>
<tr class="row-4">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Coffee Break</strong></p><br>11:00 AM - 11:20 AM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-5">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>Special Panel Session on Genomic Privacy</strong></p><br><p style="display:inline; color: #c8b18b">11:20 AM - 12:30 PM</p><br><p style="display:inline; color: #c8b18b"><strong>Chair:</strong> Bonnie Berger</p></center></td>
</tr>
<tr class="row-6">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Lunch Break</strong></p><br>12:30 PM - 2:00 PM</center></td>
</tr>
<tr class="row-7">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>PROTEOMICS</strong></p><br>2:00 PM - 2:40 PM<br><strong>Chair:</strong> Knut Reinert<br><p style="display:inline;"><strong>Venue:</strong> Lisner Auditorium</p><br><br />
<p style="font-size:18px; display:inline;"><strong> De Novo Peptide Sequencing Reveals a Vast Cyclopeptidome in Human Gut and Other Environments </strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Bahar Behsaz</strong></span>, Hosein Mohimani, Alexey Gurevich, Andrey Prjibelski, Mark F Fisher, Larry Smarr, Pieter C. Dorrestein, Joshua S. Mylne and Pavel A. Pevzner<br><br />
<p style="font-size:18px; display:inline;"><strong>Deep learning enables de novo peptide sequencing from data-independent-acquisition mass spectrometry <mark>[Highlight]</mark></strong></p><br><strong>Authors:</strong> Ngoc-Hieu Tran, <span style="background-color:#c8b18b;"><strong>Rui Qiao</strong></span>, Lei Xin, Xin Chen, Chuyi Liu, Xianglilan Zhang, Baozhen Shan, Ali Ghodsi and Ming Li</center></td>
</tr>
<tr class="row-8">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>GENOME ORGANIZATION</strong></p><br>2:40 PM - 3:20 PM<br><strong>Chair:</strong> Vineet Bafna<br><br />
<p style="font-size:18px; display:inline;"><strong>Comparing 3D Genome Organization in Multiple Species using Phylo-HMRF</strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Yang Yang</strong></span>, Yang Zhang, Bing Ren, Jesse Dixon and Jian Ma<br><br />
<p style="font-size:18px; display:inline;"><strong>ModHMM: A modular supra-Bayesian genome segmentation method</strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Philipp Benner</strong></span> and Martin Vingron</center></td>
</tr>
<tr class="row-9">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Coffee Break</strong></p><br>3:20 PM - 3:40 PM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-10">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>SEQUENCING II</strong></p><br>3:40 PM - 5:00 PM<br><strong>Chair:</strong> Pavel Avdeyev<br><br />
<p style="font-size:18px; display:inline;"><strong>An Efficient and Scalable Representation of High-Dimensional Color Information Enabled via de Bruijn Graph Search</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Fatemeh Almodaresi</strong></span>, Prashant Pandey, Michael Ferdman, Rob Johnson and Rob Patro<br><br />
<p style="font-size:18px; display:inline;"><strong>Sparse Binary Relation Representations for Genome Graph Annotation</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Mikhail Karasikov</strong></span>, Harun Mustafa, Amir Joudaki, Sara Javadzadeh No, Gunnar Rätsch and André Kahles<br><br />
<p style="font-size:18px; display:inline;"><strong>De novo clustering of long-read transcriptome data using a greedy, quality-value based algorithm</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Kristoffer Sahlin</strong></span> and Paul Medvedev<br><br />
<p style="font-size:18px; display:inline;"><strong>OMGS: Optical Map-based Genome Scaffolding</strong> </p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Weihua Pan</strong></span>, Tao Jiang and Stefano Lonardi</center></td>
</tr>
<tr class="row-11">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Depart for Social Event</strong></p><br>5:00 PM</center></td>
</tr>
</tbody>
</table>


<table id="tablepress-5" class="tablepress tablepress-id-5">
<thead>
<tr class="row-1">
	<th class="column-1"><center><p style="font-size:22px; display:inline; text-align: center"><strong><a style="color: black" id="Wed">Wednesday, May 8</a></strong></p></center></th>
</tr>
</thead>
<tbody class="row-hover">
<tr class="row-2">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>Keynote</strong></p><br><p style="font-size:20px; display:inline; color: #c8b18b"><strong>Uncovering heterogeneity in microbial communities</strong></p><br><p style="font-size:16px; display:inline; color: #c8b18b"><a style="color: #c8b18b" href="mihai-pop/">Mihai Pop</a></p><br><p style="display:inline; color: #c8b18b">9:00 AM - 10:00 AM</p><br><p style="display:inline; color: #c8b18b"><strong>Chair:</strong> Tandy Warnow (UIUC)</p><br><p style="display:inline; color: #c8b18b"><strong>Venue:</strong> Lisner Auditorium</p></center></td>
</tr>
<tr class="row-3">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>METAGENOMICS</strong></p><br>10:00 AM - 10:40 AM<br><strong>Chair:</strong> Alex Zelikovsky<br><br />
<p style="font-size:18px; display:inline;"><strong>Minerva: An Alignment and Reference Free Approach to Deconvolve Linked-Reads for Metagenomics <mark>[Highlight]</mark></strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>David Danko</strong></span>, Dmitrii Meleshko, Daniela Bezdan, Christopher Mason and Iman Hajirasouliha<br><br />
<p style="font-size:18px; display:inline;"><strong>Fast Approximation of Frequent k-mers and Applications to Metagenomics</strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Leonardo Pellegrina</strong></span>, Cinzia Pizzi and Fabio Vandin</center></td>
</tr>
<tr class="row-4">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Coffee Break</strong></p><br>10:40 AM - 11:00 AM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-5">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>PANEL WITH NSF AND NIH PROGRAM DIRECTORS</strong></p><br><p style="display:inline; color: #c8b18b">11:00 AM - 12:00 PM</p></center></td>
</tr>
<tr class="row-6">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Lunch Break</strong></p><br>12:00 PM - 1:30 PM</center></td>
</tr>
<tr class="row-7">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>NETWORKS AND PATHWAY ANALYSIS</strong></p><br>1:30 PM - 2:30 PM<br><strong>Chair:</strong> Carl Kingsford<br><br />
<p style="font-size:18px; display:inline;"><strong>Synthesizing Signaling Pathways from Temporal Phosphoproteomic Data <mark>[Highlight]</mark> </strong> </p><br><strong>Authors:</strong> Ali Sinan Köksal, Kirsten Beck, Dylan R Cronin, Aaron McKenna, Nathan D Camp, Saurabh Srivastava, Matthew E MacGilvray, Rastislav Bodík, Alejandro Wolf-Yadlin, Ernest Fraenkel, Jasmin Fisher and <span style="background-color:#c8b18b;"><strong>Anthony Gitter</strong></span><br><br />
<p style="font-size:18px; display:inline;"><strong>GRep: Gene Set Representation via Gaussian Embedding</strong></p><br><strong>Authors:</strong> Sheng Wang, <span style="background-color:#c8b18b;"><strong>Emily Flynn</strong></span> and Russ Altman<br><br />
<p style="font-size:18px; display:inline;"><strong>The Distance Precision Matrix: computing networks from non-linear relationships <mark>[Highlight]</mark></strong> </p><br><strong>Authors:</strong> Mahsa Ghanbari, Julia Lasserre and <span style="background-color:#c8b18b;"><strong>Martin Vingron</strong></span></center></td>
</tr>
<tr class="row-8">
	<td class="column-1"><center><p style="font-size:18px; display:inline;"><strong>Coffee Break</strong></p><br>2:30 PM - 2:50 PM<br><strong>Venue:</strong> Lisner Auditorium</center></td>
</tr>
<tr class="row-9">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>ALGORITHMIC FOUNDATIONS</strong></p><br>2:50 PM - 3:30 PM<br><strong>Chair:</strong> Yann Ponty<br><br />
<p style="font-size:18px; display:inline;"><strong>APPLES: Fast Distance Based Phylogenetic Placement</strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Metin Balaban</strong></span>, Shahab Sarmashghi and Siavash Mirarab<br><br />
<p style="font-size:18px; display:inline;"><strong>On the Complexity of Sequence to Graph Alignment</strong></p><br><strong>Authors:</strong> Chirag Jain, <span style="background-color:#c8b18b;"><strong>Haowen Zhang</strong></span>, Yu Gao and Srinivas Aluru</center></td>
</tr>
<tr class="row-10">
	<td class="column-1"><center><p style="font-size:20px; display:inline;"><strong>SEQUENCE LEARNING</strong></p><br>3:30 PM - 4:10 PM<br><strong>Chair:</strong> Damian Wojtowicz<br><br />
<p style="font-size:18px; display:inline;"><strong>Learning Robust Multi-Label Sample Specific Distances for Identifying HIV-1 Drug Resistance</strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Lodewijk Brand</strong></span>, Xue Yang, Liu Kai, Saad Elbeleidy, Hua Wang and Hao Zhang<br><br />
<p style="font-size:18px; display:inline;"><strong>Biological Sequence Modeling with Convolutional Kernel Networks</strong></p><br><strong>Authors:</strong> <span style="background-color:#c8b18b;"><strong>Dexiong Chen</strong></span>, Laurent Jacob and Julien Mairal</center></td>
</tr>
<tr class="row-11">
	<td class="column-1"><center><p style="font-size:18px; display:inline; color: #c8b18b"><strong>CONFERENCE CLOSING AND AWARDS CEREMONY</strong></p><br><p style="display:inline; color: #c8b18b">4:10 PM - 5:00 PM</p></center></td>
</tr>
</tbody>
</table>

							</div>

						</article>  

						
					</div><!-- row -->           

				</div><!-- container -->

				<hr class="bottom-dashed">        

			</div>
			
		</section> 

	</main>


	<script type='text/javascript'>
/* <![CDATA[ */
var wpcf7 = {"apiSettings":{"root":"https:\/\/recomb2019.org\/wp-json\/contact-form-7\/v1","namespace":"contact-form-7\/v1"},"cached":"1"};
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/plugins/contact-form-7/includes/js/scripts.js?ver=5.1.1'></script>
<script type='text/javascript' src='https://www.google.com/recaptcha/api.js?render=6LdXS4AUAAAAAFrqX0iBbs1o5KXYzz9oa82yoWbq&#038;ver=3.0'></script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/plugins.js?ver=6.3.5'></script>
<script type='text/javascript'>
/* <![CDATA[ */
var retro_home_vars = {"speed":"500","pausetime":"4000","autosliding":"slider-auto-sliding","pausehover":"slider-pause-hover","url":"https:\/\/recomb2019.org\/wp-admin\/admin-ajax.php","ref":"12af7bd083","articlenr":"3","portfolionr":"8"};
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/retro.js?ver=6.3.5'></script>
<script type='text/javascript' src='wp-includes/js/wp-embed.min.js?ver=5.0.12'></script>
<script type="text/javascript">
( function( grecaptcha, sitekey ) {

	var wpcf7recaptcha = {
		execute: function() {
			grecaptcha.execute(
				sitekey,
				{ action: 'homepage' }
			).then( function( token ) {
				var forms = document.getElementsByTagName( 'form' );

				for ( var i = 0; i < forms.length; i++ ) {
					var fields = forms[ i ].getElementsByTagName( 'input' );

					for ( var j = 0; j < fields.length; j++ ) {
						var field = fields[ j ];

						if ( 'g-recaptcha-response' === field.getAttribute( 'name' ) ) {
							field.setAttribute( 'value', token );
							break;
						}
					}
				}
			} );
		}
	};

	grecaptcha.ready( wpcf7recaptcha.execute );

	document.addEventListener( 'wpcf7submit', wpcf7recaptcha.execute, false );

} )( grecaptcha, '6LdXS4AUAAAAAFrqX0iBbs1o5KXYzz9oa82yoWbq' );
</script>

	</body>
	
</html>
<!--
Performance optimized by W3 Total Cache. Learn more: https://www.w3-edge.com/products/

Object Caching 0/186 objects using apc
Page Caching using apc (SSL caching disabled) 
Database Caching using apc (Request-wide modification query)

Served from: recomb2019.org @ 2021-05-07 09:15:08 by W3 Total Cache
-->
