<!DOCTYPE html>

<html class="no-js" lang="en-US">

	<head>

		<meta charset="UTF-8">
		<meta http-equiv="X-UA-Compatible" content="IE=edge">
		<meta name="viewport" content="width=device-width, initial-scale=1.0">

		
		
				

		<title>MATERIALS &#8211; RECOMB-CCB 2019</title>
<link rel='dns-prefetch' href='//www.google.com' />
<link rel='dns-prefetch' href='//fonts.googleapis.com' />
<link rel='dns-prefetch' href='//s.w.org' />
<link rel="alternate" type="application/rss+xml" title="RECOMB-CCB 2019 &raquo; Feed" href="feed/" />
<link rel="alternate" type="application/rss+xml" title="RECOMB-CCB 2019 &raquo; Comments Feed" href="comments/feed/" />
		<script type="text/javascript">
			window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11.2.0\/72x72\/","ext":".png","svgUrl":"https:\/\/s.w.org\/images\/core\/emoji\/11.2.0\/svg\/","svgExt":".svg","source":{"concatemoji":"https:\/\/recombccb.recomb2019.org\/wp-includes\/js\/wp-emoji-release.min.js?ver=5.1.8"}};
			!function(a,b,c){function d(a,b){var c=String.fromCharCode;l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,a),0,0);var d=k.toDataURL();l.clearRect(0,0,k.width,k.height),l.fillText(c.apply(this,b),0,0);var e=k.toDataURL();return d===e}function e(a){var b;if(!l||!l.fillText)return!1;switch(l.textBaseline="top",l.font="600 32px Arial",a){case"flag":return!(b=d([55356,56826,55356,56819],[55356,56826,8203,55356,56819]))&&(b=d([55356,57332,56128,56423,56128,56418,56128,56421,56128,56430,56128,56423,56128,56447],[55356,57332,8203,56128,56423,8203,56128,56418,8203,56128,56421,8203,56128,56430,8203,56128,56423,8203,56128,56447]),!b);case"emoji":return b=d([55358,56760,9792,65039],[55358,56760,8203,9792,65039]),!b}return!1}function f(a){var c=b.createElement("script");c.src=a,c.defer=c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var g,h,i,j,k=b.createElement("canvas"),l=k.getContext&&k.getContext("2d");for(j=Array("flag","emoji"),c.supports={everything:!0,everythingExceptFlag:!0},i=0;i<j.length;i++)c.supports[j[i]]=e(j[i]),c.supports.everything=c.supports.everything&&c.supports[j[i]],"flag"!==j[i]&&(c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&c.supports[j[i]]);c.supports.everythingExceptFlag=c.supports.everythingExceptFlag&&!c.supports.flag,c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.everything||(h=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",h,!1),a.addEventListener("load",h,!1)):(a.attachEvent("onload",h),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),g=c.source||{},g.concatemoji?f(g.concatemoji):g.wpemoji&&g.twemoji&&(f(g.twemoji),f(g.wpemoji)))}(window,document,window._wpemojiSettings);
		</script>
		<style type="text/css">
img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}
</style>
	<link rel='stylesheet' id='wp-block-library-css'  href='wp-includes/css/dist/block-library/style.min.css?ver=5.1.8' type='text/css' media='all' />
<link rel='stylesheet' id='contact-form-7-css'  href='wp-content/plugins/contact-form-7/includes/css/styles.css?ver=5.1.1' type='text/css' media='all' />
<link rel='stylesheet' id='retro-style-css'  href='wp-content/themes/Retro-4/style.css?ver=5.1.8' type='text/css' media='all' />
<link rel='stylesheet' id='parent-style-css'  href='wp-content/themes/Retro-4/style.css?ver=5.1.8' type='text/css' media='all' />
<link rel='stylesheet' id='child-style-css'  href='wp-content/themes/Retro-4-child/style.css?ver=1554754271' type='text/css' media='all' />
<link rel='stylesheet' id='retro-fonts-css'  href='https://fonts.googleapis.com/css?family=Roboto+Slab%3A400%2C700%7COswald%3A400%2C700&#038;subset&#038;ver=5.1.8' type='text/css' media='all' />
<link rel='stylesheet' id='retro-icons-css'  href='wp-content/themes/Retro-4/fonts/retro-icons.css?ver=5.1.8' type='text/css' media='all' />
<link rel='stylesheet' id='tablepress-default-css'  href='wp-content/plugins/tablepress/css/default.min.css?ver=1.9.2' type='text/css' media='all' />
<script type='text/javascript' src='wp-includes/js/jquery/jquery.js?ver=1.12.4'></script>
<script type='text/javascript' src='wp-includes/js/jquery/jquery-migrate.min.js?ver=1.4.1'></script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/modernizr.js?ver=2.7.1'></script>
<link rel='https://api.w.org/' href='wp-json/' />
<link rel="EditURI" type="application/rsd+xml" title="RSD" href="xmlrpc.php?rsd" />
<link rel="wlwmanifest" type="application/wlwmanifest+xml" href="wp-includes/wlwmanifest.xml" /> 
<meta name="generator" content="WordPress 5.1.8" />
<link rel="canonical" href="materials/" />
<link rel='shortlink' href='?p=123' />
<link rel="alternate" type="application/json+oembed" href="wp-json/oembed/1.0/embed?url=https%3A%2F%2Frecombccb.recomb2019.org%2Fmaterials%2F" />
<link rel="alternate" type="text/xml+oembed" href="wp-json/oembed/1.0/embed?url=https%3A%2F%2Frecombccb.recomb2019.org%2Fmaterials%2F&#038;format=xml" />
<style>body, input, textarea, .blog-list .post-title h3, .blog-list h4.post-meta, .portfolio-list ul li h3, .single h2.post-title, .page h2.post-title { font-family: "Roboto Slab", sans-serif; }</style><style>.hentry-content h1, .hentry-content h2, .hentry-content h3, .hentry-content h4, .hentry-content h5, .hentry-content h6, .section-subtitle, .blog-list h4.post-meta, .single h4.post-meta, .widget h3, .comments h3, .comments h4, .comment-author, .more-posts { font-family: "Oswald", serif; }</style>		<style type="text/css">.recentcomments a{display:inline !important;padding:0 !important;margin:0 !important;}</style>
		<link rel="icon" href="wp-content/uploads/2018/12/RECOMB-2019-thumbnail-80x80.png" sizes="32x32" />
<link rel="icon" href="wp-content/uploads/2018/12/RECOMB-2019-thumbnail.png" sizes="192x192" />
<link rel="apple-touch-icon-precomposed" href="wp-content/uploads/2018/12/RECOMB-2019-thumbnail.png" />
<meta name="msapplication-TileImage" content="wp-content/uploads/2018/12/RECOMB-2019-thumbnail.png" />
		<style type="text/css" id="wp-custom-css">
			.sponsors-list {
	display: flex;
	flex-wrap: wrap;
	justify-content: center;
	align-items: center;
	margin: 0 auto;
	padding: 1rem;
}

.sponsors-list a {
	vertical-align: middle;
	padding: 1rem 1rem;
	max-width: 220px;
	text-decoration:none!important;
}

.sponsors-list img {
	width: 100%
}

.row_picture {
  display: flex;
}

.column_picture {
  flex: 50%;
  padding: 5px;
}
		</style>
		
		<style>

@font-face {
  font-family: "BazarMedium";
  src: url("wp-content/themes/Retro-4/fonts/bazarmedium.eot");
  src: url("wp-content/themes/Retro-4/fonts/bazarmedium.eot?#iefix") format("embedded-opentype"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.woff") format("woff"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.ttf") format("truetype"),
       url("wp-content/themes/Retro-4/fonts/bazarmedium.svg#BazarMedium") format("svg");
  font-weight: normal;
  font-style: normal;
}

/* Fix ugly font rendering */
@media screen and (-webkit-min-device-pixel-ratio:0) {
    @font-face {
        font-family: "BazarMedium";
            src: url("wp-content/themes/Retro-4/fonts/bazarmedium.svg#BazarMedium") format("svg");
    }
}

h1, h2, h3, h4, h5, h6,
nav ul#main-nav li {
  font-family: "BazarMedium";
  text-transform: uppercase;
}




	#logo a {

		background-image: url(wp-content/uploads/2019/02/recomb-satellite-2-2_CCB-gold.png);

	}






@import url('http://fonts.googleapis.com/css?family=PT+Sans:400,700');

.sticky-note{
  overflow-y: auto;
}
</style>
	</head>

	<body id="start" class="page-template-default page page-id-123 fixed-header left-logo">
<header role="banner">

    <div class="header-inner " style="background-color: #004065">

        <div class="container">

            <div class="row clear">

                
<div id="logo">

	<a href="index.html" title="RECOMB-CCB 2019" >

		<h1>RECOMB-CCB 2019</h1>

	</a>
	
</div>
                <nav role="navigation">

                    <ul id="main-nav" class="clear"><li id="menu-item-54" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="keynotes.html">KEYNOTES</a></li>
<li id="menu-item-162" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="schedule.html">SCHEDULE</a></li>
<li id="menu-item-126" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="materials.html">MATERIALS</a></li>
<li id="menu-item-205" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="abstracts.html">ABSTRACTS</a></li>
<li id="menu-item-237" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="venue.html">VENUE</a></li>
<li id="menu-item-55" class="menu-item menu-item-type-post_type menu-item-object-page"><a href="committees.html">COMMITTEES</a></li>
<li id="menu-item-8" class="menu-item menu-item-type-custom menu-item-object-custom"><a href="../index.html">RECOMB</a></li>
</ul>

                </nav>

            </div><!-- row -->

        </div><!-- container -->

        <hr class="bottom-dashed">  

    </div><!-- header-inner -->   

</header>
			

	
	<main role="main">

		<section>

			<div class="section-inner section-post " style="background-color: #fffffe">
				
				<hr class="top-dashed"> 

				<div class="container">

					<div class="row clear">

						<article class="col col-12 tablet-full mobile-full">

							<h2 class="post-title">MATERIALS</h2>

							<hr class="single-hr">

							<div class="hentry-content">
								<p style="text-align: center;"><a href="schedule.html">RECOMB-CCB 2019 Schedule is avaliable here.</a></p>
<ul>
<li><a href="#Key">Keynote Talks</a></li>
<li><a href="#Acc">Accepted Papers</a></li>
<li><a href="#High">Highlight Talks</a></li>
<li><a href="#LBR">Late Breaking Research Talks</a></li>
<li><a href="#AccPost">Accepted Posters</a></li>
</ul>
<p><center></p>
<h2 id="Key">Keynote Talks</h2>
<p></center></p>
<ol>
<li>
<div class="paper"><span class="authors">Christina Leslie</span><span class="title"><i>Decoding epigenetic states of immune and cancer cells</i></span></li>
<li>
<div class="paper"><span class="authors">Eytan Ruppin</span><span class="title"><i>TBA</i></span></li>
<li>
<div class="paper"><span class="authors">Joel Saltz</span><span class="title"><i>Pathomics, Tumor Infiltrating Lymphocytes, Clinical Studies and Cancer Surveillance</i></span></li>
</ol>
<p><center></p>
<h2 id="Acc">Accepted Papers</h2>
<p></center></p>
<ol>
<li>
<div class="paper"><span class="authors">Hossein Sharifi-Noghabi, Yang Liu, Nicholas Erho, Raunak Shrestha, Mohammed Alshalalfa, Elai Davicioni, Colin Collins and Martin Ester.</span><span class="title"><i>Deep Genomic Signature for early metastasis prediction in prostate cancer</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Fang Wang, Shaoheng Liang, Tapsi Kumar, Nicholas Navin and Ken Chen.</span><span class="title"><i>SCMarker: ab initio marker selection for single cell transcriptome profiling</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Sean West, Hesham Ali and Dario Ghersi.</span><span class="title"><i>Uncovering and characterizing splice variants associated with survival in lung cancer patients</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Zach DiNardo, Kiran Tomlinson, Anna Ritz and Layla Oesper.</span><span class="title"><i>Distance Measures for Tumor Evolutionary Trees</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Sahand Khakabimamaghani, Dujian Ding, Oliver Snow and Martin Ester.</span><span class="title"><i>Uncovering the subtype-specific temporal order of cancer pathway dysregulation</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Pavel Skums, Viachaslau Tsyvina and Alex Zelikovsky.</span><span class="title"><i>SCIFIL: analysis of evolutionary history and clonal selection in cancer populations using single-cell sequencing data</i></span></div>
</li>
</ol>
<p><center></p>
<h2 id="High">Highlight Talks</h2>
<p></center></p>
<ol>
<li>
<div class="paper"><span class="authors">Tengfei Xiao, Wei Li, Shirley Liu and Myles Brown.</span><span class="title"><i>An Estrogen Regulated Feedback Loop Limits the Efficacy of Estrogen Receptor Targeted Breast Cancer Therapy</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Joo Sang Lee, Lital Adler, Hiren Karathia, Narin Carmel, Shiran Rabinovich, Noam Auslander, Rom Keshet, Arie Admon, David Wilson Iii, Yardena Samuels, Sridhar Hannenhalli, Eytan Ruppin and Ayelet Erez.</span><span class="title"><i>Urea Cycle Dysregulation Generates Clinically Relevant Genomic and Biochemical Signatures</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">A Forget, L Martignetti, S Puget, L Calzone, S Brabetz, D Picard, A Montagud, S Liva, A Sta, F Dingli, G Arras, J Rivera, D Loew, A Besnard, J Lacombe, M Pagès, P Varlet, C Dufour, H Yu, Al Mercier, E Indersie, A Chivet, S Leboucher, L Sieber, K Beccaria, M Gombert, Fd Meyer, N Qin, J Bartl, L Chavez, K Okonechnikov, T Sharma, V Thatikonda, F Bourdeaut, C Pouponnot, V Ramaswamy, A Korshunov, A Borkhardt, G Reifenberger, P Poullet, Md Taylor, M Kool, Sm Pfister, D Kawauchi, Emmanuel Barillot, M Remke and O Ayrault.</span><span class="title"><i>Aberrant ERBB4-SRC Signaling as a Hallmark of Group 4 Medulloblastoma Revealed by Integrative Phosphoproteomic Profiling</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Violeta Beleva Guthrie, Lily Zheng, Cathy Guerra, Yuko Kuboki, Laura Wood and Rachel Karchin.</span><span class="title"><i>Inference of single-cell genotypes characterizes the genetic heterogeneity in pancreatic cancer precursor lesions</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Johannes G. Reiter, Alvin P. Makohon-Moore, Jeffrey M. Gerold, Alexander Heyde, Christine A. Iacobuzio-Donahue, Bert Vogelstein and Martin A. Nowak.</span><span class="title"><i>Minimal functional driver gene heterogeneity among untreated metastases</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Fang Wang, Shaojun Zhang, Tae-Beom Kim, Yu-Yu Lin, Ramiz Iqbal, Zixing Wang, Vakul Mohanty, Kanishka Sircar, Jose Karam, Michael Wendl, Funda Meric-Bernstam, John Weinstein, Li Ding, Gordon Mills and Ken Chen.</span><span class="title"><i>Integrated Transcriptomic-Genomic tool Texomer profiles cancer tissues</i></span></div>
</li>
</ol>
<p><center></p>
<h2 id="LBR">Late Breaking Research Talks</h2>
<p></center></p>
<ol>
<li>
<div class="paper"><span class="authors">Nuraini Aguse, Yuanyuan Qi and Mohammed El-Kebir.</span><span class="title"><i>Summarizing the Solution Space in Tumor Phylogeny Inference by Multiple Consensus Trees</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Hatice Osmanbeyoglu, Fumiko Shimizu, Angela Rynne-Vidal, Tsz-Lun Yeung, Hannah Wen, Petar Jelinic, Samuel Mok, Gabriela Chiosis, Douglas Levine and Christina Leslie.</span><span class="title"><i>Chromatin-informed inference of transcriptional programs in gynecologic and basal breast cancers</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Gryte Satas, Simone Zaccaria, Geoffrey Mon and Ben Raphael.</span><span class="title"><i>Single-cell phylogenetic inference with copy-number supported mutation losses</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Welles Robinson, Roded Sharan and Mark DM Leiserson.</span><span class="title"><i>Modeling Clinical and Molecular Covariates of Mutational Process Activity in Cancer</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Yoo-Ah Kim, Rebecca Sarto Basso, Damian Wójtowicz, Dorit S. Hochbaum, Fabio Vandin and Teresa Przytycka.</span><span class="title"><i>Identifying Drug Sensitivity Subnetworks with NETPHLIX</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Xiaoshan Melody Shao, Justin Huang, Ashok Sivakumar, Kymberleigh Pagel and Rachel Karchin.</span><span class="title"><i>High-throughput binding prediction of Class I and Class II neoantigens with MHCnuggets</i></span></div>
</li>
</ol>
<p><center></p>
<h2 id="AccPost">Accepted Posters</h2>
<p></center></p>
<ol>
<li>
<div class="paper"><span class="authors">Joo Sang Lee, Avinash Das, Livnat Jerby-Arnon, Rand Arafeh, Noam Auslander, Matthew Davidson, Lynn McGarry, Daniel James, Arnaud Amzallag, Seung Gu Park, Kuoyuan Cheng, Welles Robinson, Dikla Atias, Chani Stossel, Ella Buzhor, Gidi Stein, Joshua Waterfall, Paul Meltzer, Talia Golan, Sridhar Hannenhalli, Eyal Gottlieb, Cyril Benes, Yardena Samuels, Emma Shanks and Eytan Ruppin.</span><span class="title"><i>Harnessing synthetic lethality to predict the response to cancer treatment</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Michael Bradshaw and Samuel Payne.</span><span class="title"><i>Detection of Fraud in Large Omics Datasets</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Collin Tokheim and Rachel Karchin.</span><span class="title"><i>CHASMplus reveals the scope of somatic missense mutations driving human cancers</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Inna Kuperstein.</span><span class="title"><i>Atlas of Cancer Signaling Network: a resource of multi-scale biological maps to study disease mechanisms</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Inna Kuperstein.</span><span class="title"><i>Comprehensive map of regulated cell death signalling network: a powerful analytical tool for studying diseases</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Chunhui Cai, Gregory Cooper, Kevin Lu, Xiaojun Ma, Shuping Xu, Zhenlong Zhao, Xueer Chen, Yifan Xue, Adrian Lee, Nathan Clark, Vicky Chen, Songjian Lu, Lujia Chen, Liyue Yu, Harry Hochheiser, Xia Jiang, Jane Wang and Xinghua Lu.</span><span class="title"><i>Systematic Discovery of the Functional Impact of Somatic Genome Alterations in Individual Tumors through Tumor-specific Causal Inference</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Yangqiaoyu Zhou and Layla Oesper.</span><span class="title"><i>A Topology-Aware Edit Distance Measure For Cancer Evolutionary Trees</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Simone Zaccaria and Ben Raphael.</span><span class="title"><i>Accurate quantification of copy-number aberrations and whole-genome duplications in multi-sample tumor sequencing data</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Simone Ciccolella, Murray Patterson, Paola Bonizzoni and Gianluca Della Vedova.</span><span class="title"><i>Effective clustering for single cell sequencing cancer data</i></span></div>
</li>
<li value="11">
<div class="paper"><span class="authors">Qi Miao, Fang Wang, Rafet Basar, Muharrem Muftuoglu, Li Li, Katy Rezvani and Ken Chen.</span><span class="title"><i>Ab initio Spillover Compensation in CyTOF Data</i></span></div>
</li>
<li value="12">
<div class="paper"><span class="authors">Harshit Sahay, Ariel Afek and Raluca Gordan.</span><span class="title"><i>Toward understanding mutagenesis in transcription factor binding sites</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Shaoheng Liang, Fang Wang, Jincheng Han and Ken Chen.</span><span class="title"><i>Latent periodic process inference from single-cell RNA-seq data</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Coby Viner, James Johnson, Laura García-Prat, Charles A. Ishak, Nicolas Walker, Hui Shi, Marcela Sjöberg-Herrera, Shu Yi Shen, David J. Adams, John E. Dick, Anne C. Ferguson-Smith, Daniel D. De Carvalho, Timothy L. Bailey and Michael M. Hoffman.</span><span class="title"><i>Methyl-sensitive transcription factor motifs in leukemia cell populations</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Fang Wang, Qihan Wang, Jincheng Han, Shaoheng Liang, Ruli Gao, Li Ding, Nicholas Navin and Ken Chen.</span><span class="title"><i>Tracing tumor cellular evolution through copy number alterations</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Nishanth Ulhas Nair, Kuoyuan Cheng, Joo Sang Lee and Eytan Ruppin.</span><span class="title"><i>Contribution of synthetic lethality to cancer risk and onset time across human tissues</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Kuoyuan Cheng, Sridhar Hannenhalli and Eytan Ruppin.</span><span class="title"><i>Integrating (phospho)proteomics for improved identification of signal pathway abnormalities in cancer</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Yoo-Ah Kim, Damian Wójtowicz, Rebecca Sarto Basso, Itay Sason, Welles Robinson, Dorit S. Hochbaum, Mark DM Leiserson, Roded Sharan, Fabio Vandin and Teresa Przytycka.</span><span class="title"><i>Network-based approaches elucidate differences within APOBEC and clock-like signatures in breast cancer</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Amanda Oliphant, Emily Hoskins, Samuel Pugh, Jonathan Jarman, Daniel Cui Zhou, David Adams, Sean Beecroft, Li Ding and Samuel Payne.</span><span class="title"><i>Analysis of clustered mutations on highly mutated proteins in endometrial cancer</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Ashok Sivakumar, Dylan Hirsch, Rohit Bhattacharya, Justin Huang, Collin Tokheim, Valsamo Anagnostou, Victor Velculescu, Simon Wing and Rachel Karchin.</span><span class="title"><i>Information Theoretic approaches to interrogate T-cell receptor diversity for cancer immunotherapy</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Kymberleigh Pagel, Lily Zheng, Rick Kim, Kyle Moad, Michael Ryan and Rachel Karchin.</span><span class="title"><i>The Open Custom-Ranked Analysis of Variants Toolkit (OpenCRAVAT): A customizable annotation and prioritization pipeline for genes and variants</i></span></div>
</li>
<li>
<div class="paper"><span class="authors">Mark Keller, Welles Robinson, Mark Leiserson.</span><span class="title"><i>ExploSig: Hypothesis-driven Exploration of Mutation Signature Etiology</i></span></div>
</li>
</ol>
<p><!---

<li> 

<div class="paper"><span class="authors">Mohammed El-Kebir, Simone Zaccaria and Ben Raphael.</span><span class="title"><i>Descendant Cell Fraction: Copy-aware Inference of Clonal Composition and Evolution in Cancer</i></span></div>

</li>

----></p>
							</div>

						</article>  

						
					</div><!-- row -->           

				</div><!-- container -->

				<hr class="bottom-dashed">        

			</div>
			
		</section> 

	</main>


	<script type='text/javascript'>
/* <![CDATA[ */
var wpcf7 = {"apiSettings":{"root":"https:\/\/recombccb.recomb2019.org\/wp-json\/contact-form-7\/v1","namespace":"contact-form-7\/v1"},"cached":"1"};
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/plugins/contact-form-7/includes/js/scripts.js?ver=5.1.1'></script>
<script type='text/javascript' src='https://www.google.com/recaptcha/api.js?render=6LdXS4AUAAAAAFrqX0iBbs1o5KXYzz9oa82yoWbq&#038;ver=3.0'></script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/plugins.js?ver=6.0.0'></script>
<script type='text/javascript'>
/* <![CDATA[ */
var retro_home_vars = {"speed":"500","pausetime":"4000","autosliding":"slider-auto-sliding","pausehover":"slider-pause-hover","url":"https:\/\/recombccb.recomb2019.org\/wp-admin\/admin-ajax.php","ref":"a2521aaaaf","articlenr":"3","portfolionr":"8"};
/* ]]> */
</script>
<script type='text/javascript' src='wp-content/themes/Retro-4/js/retro.js?ver=6.0.0'></script>
<script type='text/javascript' src='wp-includes/js/wp-embed.min.js?ver=5.1.8'></script>
<script type="text/javascript">
( function( grecaptcha, sitekey ) {

	var wpcf7recaptcha = {
		execute: function() {
			grecaptcha.execute(
				sitekey,
				{ action: 'homepage' }
			).then( function( token ) {
				var forms = document.getElementsByTagName( 'form' );

				for ( var i = 0; i < forms.length; i++ ) {
					var fields = forms[ i ].getElementsByTagName( 'input' );

					for ( var j = 0; j < fields.length; j++ ) {
						var field = fields[ j ];

						if ( 'g-recaptcha-response' === field.getAttribute( 'name' ) ) {
							field.setAttribute( 'value', token );
							break;
						}
					}
				}
			} );
		}
	};

	grecaptcha.ready( wpcf7recaptcha.execute );

	document.addEventListener( 'wpcf7submit', wpcf7recaptcha.execute, false );

} )( grecaptcha, '6LdXS4AUAAAAAFrqX0iBbs1o5KXYzz9oa82yoWbq' );
</script>

	</body>
	
</html>
<!--
Performance optimized by W3 Total Cache. Learn more: https://www.w3-edge.com/products/

Object Caching 33/38 objects using apc
Page Caching using apc 
Database Caching 1/5 queries in 0.005 seconds using apc

Served from: recombccb.recomb2019.org @ 2021-04-14 00:17:27 by W3 Total Cache
-->
